# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-04-13 11:10+0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: apps/api/base.py:229
#, python-brace-format
msgid "[{module}-API]{error_message}"
msgstr ""

#: apps/api/base.py:298
msgid "返回数据格式不正确，结果格式非json."
msgstr ""

#: apps/api/base.py:367
#, python-brace-format
msgid "[BKLOGAPI] {info}"
msgstr ""

#: apps/api/base.py:787
msgid "请求缺少org_id"
msgstr ""

#: apps/api/modules/bk_itsm.py:29
msgid "ITSM"
msgstr ""

#: apps/api/modules/bk_log.py:29
msgid "log_search元数据"
msgstr ""

#: apps/api/modules/bk_log.py:36
msgid "查询数据"
msgstr ""

#: apps/api/modules/bk_log.py:45
msgid "拉取索引mapping"
msgstr ""

#: apps/api/modules/bk_log.py:53
msgid "查询数据DSL模式"
msgstr ""

#: apps/api/modules/bk_log.py:61
msgid "scroll滚动查询"
msgstr ""

#: apps/api/modules/bk_log.py:69
msgid "获取索引列表"
msgstr ""

#: apps/api/modules/bk_log.py:77
msgid "获取集群信息"
msgstr ""

#: apps/api/modules/bk_log.py:86
msgid "获取kafka采样"
msgstr ""

#: apps/api/modules/bk_log.py:93
msgid "es请求转发"
msgstr ""

#: apps/api/modules/bk_log.py:101
msgid "连通性测试"
msgstr ""

#: apps/api/modules/bk_login.py:54
msgid "PaaS平台登录模块"
msgstr ""

#: apps/api/modules/bk_paas.py:29
msgid "PaaS平台"
msgstr ""

#: apps/api/modules/bkdata_access.py:32
msgid "数据平台接入模块"
msgstr ""

#: apps/api/modules/bkdata_aiops.py:28
msgid "数据平台aiops模块"
msgstr ""

#: apps/api/modules/bkdata_auth.py:32
msgid "数据平台鉴权模块"
msgstr ""

#: apps/api/modules/bkdata_databus.py:29
msgid "计算平台总线模块"
msgstr ""

#: apps/api/modules/bkdata_dataflow.py:27
msgid "数据平台dataflow模块"
msgstr ""

#: apps/api/modules/bkdata_meta.py:29
msgid "计算平台元数据模块"
msgstr ""

#: apps/api/modules/bkdata_query.py:30
msgid "计算平台数据查询模块"
msgstr ""

#: apps/api/modules/bkdata_storekit.py:29
msgid "数据平台存储模块"
msgstr ""

#: apps/api/modules/cc.py:49 apps/iam/handlers/permission.py:366
msgid "配置平台"
msgstr ""

#: apps/api/modules/cmsi.py:55
msgid "CMSI"
msgstr ""

#: apps/api/modules/gse.py:49
msgid "GSE"
msgstr ""

#: apps/api/modules/gse.py:56
msgid "获取agent状态"
msgstr ""

#: apps/api/modules/iam.py:32
msgid "IAM"
msgstr ""

#: apps/api/modules/job.py:31
msgid "JOB"
msgstr ""

#: apps/api/modules/job.py:37
msgid "快速执行脚本"
msgstr ""

#: apps/api/modules/job.py:44
msgid "快速分发文件"
msgstr ""

#: apps/api/modules/job.py:51
msgid "根据作业id获取执行日志"
msgstr ""

#: apps/api/modules/monitor.py:32
msgid "MONITOR"
msgstr ""

#: apps/api/modules/paascc.py:28
msgid " 蓝鲸 PaaS 3.0 配置中心"
msgstr ""

#: apps/api/modules/transfer.py:79
msgid "Transfer元数据"
msgstr ""

#: apps/api/modules/transfer.py:86
msgid "创建数据源"
msgstr ""

#: apps/api/modules/transfer.py:93
msgid "修改数据源"
msgstr ""

#: apps/api/modules/transfer.py:100
msgid "修改数据源与结果表的关系"
msgstr ""

#: apps/api/modules/transfer.py:107
msgid "创建结果表"
msgstr ""

#: apps/api/modules/transfer.py:114
msgid "删除存储集群"
msgstr ""

#: apps/api/modules/transfer.py:121
msgid "修改结果表"
msgstr ""

#: apps/api/modules/transfer.py:128
msgid "结果表起停"
msgstr ""

#: apps/api/modules/transfer.py:135
msgid "获取数据源标签"
msgstr ""

#: apps/api/modules/transfer.py:142
msgid "查询一个数据源的ID"
msgstr ""

#: apps/api/modules/transfer.py:149
msgid "查询一个结果表的信息"
msgstr ""

#: apps/api/modules/transfer.py:156
msgid "查询一个结果表的存储信息"
msgstr ""

#: apps/api/modules/transfer.py:164
msgid "查询存储集群列表"
msgstr ""

#: apps/api/modules/transfer.py:172 apps/api/modules/transfer.py:186
msgid "创建存储集群"
msgstr ""

#: apps/api/modules/transfer.py:179
msgid "修改存储集群"
msgstr ""

#: apps/api/modules/transfer.py:193
msgid "获取所有transfer集群信息"
msgstr ""

#: apps/api/modules/transfer.py:201
msgid "创建ES快照仓库"
msgstr ""

#: apps/api/modules/transfer.py:208
msgid "修改ES快照仓库"
msgstr ""

#: apps/api/modules/transfer.py:215
msgid "删除ES快照仓库"
msgstr ""

#: apps/api/modules/transfer.py:222
msgid "验证ES快照仓库"
msgstr ""

#: apps/api/modules/transfer.py:229
msgid "查看指定ES快照仓库"
msgstr ""

#: apps/api/modules/transfer.py:236
msgid "所有快照仓库列表"
msgstr ""

#: apps/api/modules/transfer.py:243
msgid "创建结果表快照配置"
msgstr ""

#: apps/api/modules/transfer.py:250
msgid "修改结果表快照配置"
msgstr ""

#: apps/api/modules/transfer.py:257
msgid "删除结果表快照配置"
msgstr ""

#: apps/api/modules/transfer.py:264
msgid "结果表快照配置列表"
msgstr ""

#: apps/api/modules/transfer.py:271
msgid "结果表快照配置列表含物理索引"
msgstr ""

#: apps/api/modules/transfer.py:278
msgid "获取结果表快照状态"
msgstr ""

#: apps/api/modules/transfer.py:285
msgid "快照回溯"
msgstr ""

#: apps/api/modules/transfer.py:292
msgid "修改快照回溯"
msgstr ""

#: apps/api/modules/transfer.py:299
msgid "删除快照回溯"
msgstr ""

#: apps/api/modules/transfer.py:306
msgid "快照回溯列表"
msgstr ""

#: apps/api/modules/transfer.py:313
msgid "快照回溯状态"
msgstr ""

#: apps/bk_log_admin/constants.py:36
msgid "大于30 s"
msgstr ""

#: apps/bk_log_admin/constants.py:37
msgid "10~30 s"
msgstr ""

#: apps/bk_log_admin/constants.py:38
msgid "0~10 s"
msgstr ""

#: apps/bk_log_admin/exceptions.py:27
msgid "管理端模块异常"
msgstr ""

#: apps/bk_log_admin/exceptions.py:32
msgid "检索历史不存在"
msgstr ""

#: apps/bk_log_admin/exceptions.py:37 apps/exceptions.py:80
msgid "参数验证失败"
msgstr ""

#: apps/bk_log_admin/exceptions.py:42
msgid "获取自定义上报token异常"
msgstr ""

#: apps/bk_log_admin/exceptions.py:47
msgid "获取globalConfig中custom_report_setting 异常"
msgstr ""

#: apps/bk_log_admin/exceptions.py:52
msgid "index_set_id不应该为空"
msgstr ""

#: apps/bk_log_admin/exceptions.py:57
msgid "biz_id or operate_type and operate_id 两组参数必填其中一组"
msgstr ""

#: apps/bk_log_admin/exceptions.py:62
msgid "数据源初始化异常, 请联系管理员"
msgstr ""

#: apps/bk_log_admin/handlers/audit_record.py:34
msgid "空"
msgstr ""

#: apps/bk_log_admin/handlers/audit_record.py:39
msgid "请求内容"
msgstr ""

#: apps/bk_log_admin/views/audit_record_views.py:80
#: apps/log_databus/views/collector_views.py:292
#: apps/log_extract/views/tasks_views.py:119
#: apps/log_search/views/index_set_views.py:259
msgid "分页参数不能为空"
msgstr ""

#: apps/bk_log_admin/views/index_set.py:196
#, python-brace-format
msgid "查询时间段目前只支持{constants.HISTORY_MAX_DAYS}天"
msgstr ""

#: apps/constants.py:32
msgid "邮件"
msgstr ""

#: apps/constants.py:50
msgid "腾讯云对象存储"
msgstr ""

#: apps/constants.py:50
msgid "远程文件系统"
msgstr ""

#: apps/constants.py:50
msgid "蓝鲸文件存储服务"
msgstr ""

#: apps/constants.py:71
msgid "采集项"
msgstr ""

#: apps/constants.py:72 apps/log_databus/models.py:303
msgid "存储集群"
msgstr ""

#: apps/constants.py:73 apps/log_measure/handlers/metric_collectors/index.py:31
#: apps/log_measure/handlers/metrics.py:376 apps/log_search/models.py:661
msgid "索引集"
msgstr ""

#: apps/constants.py:74
msgid "检索配置"
msgstr ""

#: apps/constants.py:75 apps/log_databus/models.py:94
#: apps/log_databus/serializers.py:647 apps/log_databus/serializers.py:654
msgid "清洗配置"
msgstr ""

#: apps/constants.py:76
msgid "导出"
msgstr ""

#: apps/constants.py:77
#: apps/log_measure/handlers/metric_collectors/log_extract.py:32
#: apps/log_measure/handlers/metrics.py:583
msgid "日志提取策略"
msgstr ""

#: apps/constants.py:78
msgid "日志提取链路"
msgstr ""

#: apps/constants.py:79
#: apps/log_measure/handlers/metric_collectors/log_extract.py:56
#: apps/log_measure/handlers/metrics.py:605 config/default.py:587
msgid "日志提取任务"
msgstr ""

#: apps/constants.py:95
msgid "创建"
msgstr ""

#: apps/constants.py:96
msgid "更新"
msgstr ""

#: apps/constants.py:97
msgid "删除"
msgstr ""

#: apps/constants.py:98
msgid "任务重试"
msgstr ""

#: apps/constants.py:99
msgid "启动"
msgstr ""

#: apps/constants.py:100
msgid "停止"
msgstr ""

#: apps/constants.py:101
msgid "新建并替换"
msgstr ""

#: apps/constants.py:102
msgid "替换"
msgstr ""

#: apps/constants.py:103
msgid "配置"
msgstr ""

#: apps/esb/exceptions.py:27
msgid "esb转发模块异常"
msgstr ""

#: apps/esb/exceptions.py:32
msgid "请检查对应请求path"
msgstr ""

#: apps/esb/exceptions.py:37
msgid "该url暂时不支持转发"
msgstr ""

#: apps/esb/exceptions.py:42
msgid "该method暂时不支持"
msgstr ""

#: apps/esb/exceptions.py:47
msgid "没有传入鉴权实例id"
msgstr ""

#: apps/exceptions.py:50
msgid "系统异常"
msgstr ""

#: apps/exceptions.py:85
msgid "远程服务请求结果异常"
msgstr ""

#: apps/exceptions.py:90
msgid "组件调用异常"
msgstr ""

#: apps/exceptions.py:95
#, python-brace-format
msgid "业务不存在: {bk_biz_id}"
msgstr ""

#: apps/exceptions.py:100
msgid "语言不支持"
msgstr ""

#: apps/exceptions.py:105
msgid "获取锁失败"
msgstr ""

#: apps/exceptions.py:110
msgid "权限不足"
msgstr ""

#: apps/exceptions.py:116
msgid "服务不稳定，请检查组件健康状况"
msgstr ""

#: apps/feature_toggle/models.py:33
msgid "特性开关name"
msgstr ""

#: apps/feature_toggle/models.py:34
msgid "特性开关别名"
msgstr ""

#: apps/feature_toggle/models.py:35
msgid "特性开关status"
msgstr ""

#: apps/feature_toggle/models.py:36
msgid "特性开关描述"
msgstr ""

#: apps/feature_toggle/models.py:37
msgid "是否在前端展示"
msgstr ""

#: apps/feature_toggle/models.py:38
msgid "特性开关配置"
msgstr ""

#: apps/feature_toggle/models.py:39
msgid "业务白名单"
msgstr ""

#: apps/feature_toggle/models.py:42
msgid "日志平台特性开关"
msgstr ""

#: apps/feature_toggle/models.py:43
msgid "41_日志平台特性开关"
msgstr ""

#: apps/generic.py:335 apps/middlewares.py:142
#, python-brace-format
msgid "【APP 自定义异常】{message}, code={code}, args={args}"
msgstr ""

#: apps/generic.py:348
msgid "系统错误，请联系管理员"
msgstr ""

#: apps/grafana/constants.py:27
msgid "主机ID"
msgstr ""

#: apps/grafana/constants.py:28 apps/grafana/constants.py:33
#: apps/grafana/constants.py:35 apps/log_databus/models.py:282
#: apps/log_databus/serializers.py:187 apps/log_databus/serializers.py:204
#: apps/log_databus/serializers.py:394 apps/log_databus/serializers.py:529
#: apps/log_search/serializers.py:58
msgid "集群ID"
msgstr ""

#: apps/grafana/constants.py:29 apps/grafana/constants.py:32
msgid "模块ID"
msgstr ""

#: apps/grafana/exceptions.py:27
msgid "Grafana 模块异常"
msgstr ""

#: apps/grafana/exceptions.py:37
msgid "创建Grafana用户失败"
msgstr ""

#: apps/grafana/exceptions.py:42
msgid "获取Grafana用户失败"
msgstr ""

#: apps/grafana/exceptions.py:47
msgid "创建Grafana组织失败"
msgstr ""

#: apps/grafana/exceptions.py:52
msgid "获取Grafana组织失败"
msgstr ""

#: apps/grafana/exceptions.py:57
msgid "获取组织中的用户失败"
msgstr ""

#: apps/grafana/handlers/home_dashboard.py:33
msgid "欢迎使用仪表盘"
msgstr ""

#: apps/grafana/handlers/home_dashboard.py:61
msgid "所有仪表盘"
msgstr ""

#: apps/grafana/handlers/home_dashboard.py:80
msgid "收藏和最近查看的仪表盘"
msgstr ""

#: apps/grafana/handlers/home_dashboard.py:108
msgid "还没有仪表盘？快速创建您的仪表盘吧！"
msgstr ""

#: apps/grafana/handlers/home_dashboard.py:110
msgid "为方便管理仪表盘，可以创建目录。"
msgstr ""

#: apps/grafana/handlers/home_dashboard.py:112
msgid "本地有仪表盘配置文件，直接导入即可。"
msgstr ""

#: apps/grafana/handlers/home_dashboard.py:114
msgid "更多使用方法查看产品文档"
msgstr ""

#: apps/grafana/handlers/home_dashboard.py:122
msgid "仪表盘使用指引"
msgstr ""

#: apps/grafana/handlers/query.py:54 apps/grafana/handlers/query.py:56
#: apps/log_search/views/search_views.py:781
#: apps/log_search/views/search_views.py:783
msgid "请输入，注意空格符号"
msgstr ""

#: apps/grafana/handlers/query.py:55 apps/grafana/handlers/query.py:57
#: apps/log_search/views/search_views.py:782
#: apps/log_search/views/search_views.py:784
msgid "逗号分隔"
msgstr ""

#: apps/grafana/handlers/query.py:58 apps/grafana/handlers/query.py:59
#: apps/grafana/handlers/query.py:60 apps/grafana/handlers/query.py:61
#: apps/log_search/views/search_views.py:785
#: apps/log_search/views/search_views.py:786
#: apps/log_search/views/search_views.py:787
#: apps/log_search/views/search_views.py:788
msgid "请输入可以支持范围过滤字段相应的值，如整数"
msgstr ""

#: apps/grafana/handlers/query.py:386
msgid "未分类"
msgstr ""

#: apps/grafana/model.py:25
msgid "trace索引集id"
msgstr ""

#: apps/grafana/model.py:26 apps/log_databus/models.py:298
#: apps/log_databus/serializers.py:636
msgid "数据源id"
msgstr ""

#: apps/grafana/provisioning.py:37
msgid "日志平台-时序数据"
msgstr ""

#: apps/grafana/serializers.py:27 apps/grafana/serializers.py:32
#: apps/grafana/serializers.py:38 apps/grafana/serializers.py:47
#: apps/grafana/serializers.py:56 apps/grafana/serializers.py:75
#: apps/grafana/serializers.py:86 apps/log_databus/serializers.py:153
#: apps/log_databus/serializers.py:157 apps/log_databus/serializers.py:179
#: apps/log_databus/serializers.py:217 apps/log_databus/serializers.py:346
#: apps/log_databus/serializers.py:351 apps/log_databus/serializers.py:395
#: apps/log_databus/serializers.py:666 apps/log_databus/serializers.py:670
#: apps/log_databus/serializers.py:674 apps/log_databus/serializers.py:683
#: apps/log_databus/serializers.py:691 apps/log_databus/serializers.py:692
#: apps/log_databus/serializers.py:709 apps/log_databus/serializers.py:715
#: apps/log_extract/models.py:45 apps/log_extract/serializers.py:60
#: apps/log_extract/serializers.py:93 apps/log_extract/serializers.py:98
#: apps/log_extract/serializers.py:103 apps/log_extract/serializers.py:247
#: apps/log_extract/serializers.py:325 apps/log_extract/serializers.py:373
#: apps/log_extract/serializers.py:381 apps/log_search/models.py:164
#: apps/log_search/models.py:575 apps/log_search/models.py:778
#: apps/log_search/serializers.py:57 apps/log_search/serializers.py:145
#: apps/log_search/serializers.py:198 apps/log_search/serializers.py:359
#: apps/log_trace/serializers.py:40 apps/log_trace/serializers.py:52
msgid "业务ID"
msgstr ""

#: apps/grafana/serializers.py:28 apps/grafana/serializers.py:33
msgid "查询类型"
msgstr ""

#: apps/grafana/serializers.py:34
msgid "查询参数"
msgstr ""

#: apps/grafana/serializers.py:39 apps/log_databus/serializers.py:186
#: apps/log_databus/serializers.py:200 apps/log_databus/serializers.py:224
msgid "分类ID"
msgstr ""

#: apps/grafana/serializers.py:48 apps/log_extract/serializers.py:382
#: apps/log_search/serializers.py:309
msgid "实例类型"
msgstr ""

#: apps/grafana/serializers.py:49 apps/log_extract/serializers.py:383
#: apps/log_search/serializers.py:310
msgid "是否删除空节点"
msgstr ""

#: apps/grafana/serializers.py:53 apps/grafana/serializers.py:72
msgid "仪表盘ID"
msgstr ""

#: apps/grafana/serializers.py:54 apps/grafana/serializers.py:73
msgid "面板ID"
msgstr ""

#: apps/grafana/serializers.py:57
msgid "监控指标"
msgstr ""

#: apps/grafana/serializers.py:58 apps/grafana/serializers.py:76
#: apps/grafana/serializers.py:87 apps/log_databus/models.py:93
#: apps/log_databus/serializers.py:525
msgid "结果表ID"
msgstr ""

#: apps/grafana/serializers.py:59 apps/grafana/serializers.py:77
msgid "过滤条件"
msgstr ""

#: apps/grafana/serializers.py:60 apps/log_clustering/models.py:72
msgid "聚合字段"
msgstr ""

#: apps/grafana/serializers.py:61 apps/grafana/serializers.py:78
#: apps/grafana/serializers.py:91
msgid "查询字符串"
msgstr ""

#: apps/grafana/serializers.py:63
msgid "聚合方法"
msgstr ""

#: apps/grafana/serializers.py:64
msgid "时间间隔"
msgstr ""

#: apps/grafana/serializers.py:65 apps/grafana/serializers.py:79
msgid "监控目标"
msgstr ""

#: apps/grafana/serializers.py:67 apps/grafana/serializers.py:81
#: apps/grafana/serializers.py:89 apps/grafana/serializers.py:101
#: apps/log_databus/models.py:382 apps/log_extract/serializers.py:65
#: apps/log_trace/constants.py:71
#: apps/log_trace/handlers/trace_field_handlers.py:44
#: apps/log_trace/handlers/trace_field_handlers.py:51
msgid "开始时间"
msgstr ""

#: apps/grafana/serializers.py:68 apps/grafana/serializers.py:82
#: apps/grafana/serializers.py:90 apps/grafana/serializers.py:102
#: apps/log_databus/models.py:383 apps/log_extract/serializers.py:66
#: apps/log_search/serializers.py:253
msgid "结束时间"
msgstr ""

#: apps/grafana/serializers.py:80
msgid "日志条数"
msgstr ""

#: apps/grafana/serializers.py:88
msgid "查询字段"
msgstr ""

#: apps/grafana/serializers.py:95
msgid "operation"
msgstr ""

#: apps/grafana/serializers.py:96
msgid "service"
msgstr ""

#: apps/grafana/serializers.py:97
msgid "tags"
msgstr ""

#: apps/grafana/serializers.py:98
msgid "minDuration"
msgstr ""

#: apps/grafana/serializers.py:99
msgid "maxDuration"
msgstr ""

#: apps/grafana/serializers.py:100
msgid "limit"
msgstr ""

#: apps/iam/exceptions.py:57
#, python-brace-format
msgid "当前用户无 [{action_name}] 权限"
msgstr ""

#: apps/iam/handlers/actions.py:213
#, python-brace-format
msgid "动作ID不存在：{action_id}"
msgstr ""

#: apps/iam/handlers/permission.py:306
#, python-brace-format
msgid "获取系统信息错误：{message}"
msgstr ""

#: apps/iam/handlers/resources.py:212
#, python-brace-format
msgid "资源ID不存在：{resource_id}"
msgstr ""

#: apps/log_audit/models.py:27
msgid "操作时间"
msgstr ""

#: apps/log_audit/models.py:28
msgid "操作者"
msgstr ""

#: apps/log_audit/models.py:29 apps/log_clustering/models.py:74
#: apps/log_clustering/models.py:93 apps/log_clustering/models.py:115
#: apps/log_databus/models.py:78 apps/log_databus/models.py:257
#: apps/log_databus/models.py:271 apps/log_databus/models.py:281
#: apps/log_databus/models.py:305 apps/log_databus/models.py:321
#: apps/log_databus/models.py:335 apps/log_databus/models.py:346
#: apps/log_databus/models.py:379 apps/log_databus/serializers.py:606
#: apps/log_databus/serializers.py:610 apps/log_databus/serializers.py:635
#: apps/log_databus/serializers.py:640 apps/log_databus/serializers.py:649
#: apps/log_databus/serializers.py:656
#: apps/log_extract/components/collections/distribution_component.py:47
#: apps/log_extract/components/collections/packing_component.py:47
#: apps/log_extract/models.py:74 apps/log_search/serializers.py:249
#: apps/log_search/serializers.py:277 apps/log_search/serializers.py:321
msgid "业务id"
msgstr ""

#: apps/log_audit/models.py:30
msgid "操作对象类型"
msgstr ""

#: apps/log_audit/models.py:31
msgid "操作对象子类型"
msgstr ""

#: apps/log_audit/models.py:32
msgid "操作对象id"
msgstr ""

#: apps/log_audit/models.py:33
msgid "操作方法"
msgstr ""

#: apps/log_audit/models.py:34
msgid "请求参数"
msgstr ""

#: apps/log_audit/models.py:37 apps/log_audit/models.py:38
msgid "操作日志"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:32
msgid "创建模型"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:67
msgid "更新持续训练"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:96
msgid "创建实验"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:131
msgid "变更实验meta配置"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:164
msgid "执行样本准备"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:225
msgid "执行样本切分"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:287
msgid "执行模型训练"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:365
msgid "执行模型评估"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:421
msgid "获取模型评估结果"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:460
msgid "执行提交"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:492
msgid "执行发布"
msgstr ""

#: apps/log_clustering/components/collections/aiops_model_component.py:530
msgid "获取pattern"
msgstr ""

#: apps/log_clustering/components/collections/data_access_component.py:34
msgid "切换数据流"
msgstr ""

#: apps/log_clustering/components/collections/data_access_component.py:67
msgid "创建清洗接入"
msgstr ""

#: apps/log_clustering/components/collections/data_access_component.py:97
msgid "同步清洗配置到数据平台"
msgstr ""

#: apps/log_clustering/components/collections/data_access_component.py:125
msgid "项目添加rt权限"
msgstr ""

#: apps/log_clustering/components/collections/flow_component.py:39
msgid "创建预处理flow"
msgstr ""

#: apps/log_clustering/components/collections/flow_component.py:82
msgid "创建After处理flow"
msgstr ""

#: apps/log_clustering/components/collections/flow_component.py:108
#: apps/log_databus/models.py:415
msgid "【日志平台】"
msgstr ""

#: apps/log_clustering/components/collections/flow_component.py:133
msgid "创建新类策略"
msgstr ""

#: apps/log_clustering/components/collections/sample_set_component.py:32
msgid "创建样本集"
msgstr ""

#: apps/log_clustering/components/collections/sample_set_component.py:69
msgid "把rt添加到stag表"
msgstr ""

#: apps/log_clustering/components/collections/sample_set_component.py:107
msgid "创建或更新样本采集配置"
msgstr ""

#: apps/log_clustering/components/collections/sample_set_component.py:132
msgid "执行样本集提交"
msgstr ""

#: apps/log_clustering/constants.py:61
msgid "日志平台日志聚类告警"
msgstr ""

#: apps/log_clustering/constants.py:76
msgid "日志聚类新类(近24H)"
msgstr ""

#: apps/log_clustering/constants.py:143
msgid "不比对"
msgstr ""

#: apps/log_clustering/constants.py:144
msgid "1小时前"
msgstr ""

#: apps/log_clustering/constants.py:145
msgid "2小时前"
msgstr ""

#: apps/log_clustering/constants.py:146
msgid "3小时前"
msgstr ""

#: apps/log_clustering/constants.py:147
msgid "6小时前"
msgstr ""

#: apps/log_clustering/constants.py:148
msgid "12小时前"
msgstr ""

#: apps/log_clustering/constants.py:149
msgid "24小时前"
msgstr ""

#: apps/log_clustering/exceptions.py:32
msgid "日志聚类模块异常"
msgstr ""

#: apps/log_clustering/exceptions.py:37
msgid "聚类未开放"
msgstr ""

#: apps/log_clustering/exceptions.py:42
#, python-brace-format
msgid "获取node config异常{steps}"
msgstr ""

#: apps/log_clustering/exceptions.py:47
#, python-brace-format
msgid "不支持的step_name状态获取: {step_name}"
msgstr ""

#: apps/log_clustering/exceptions.py:52
#, python-brace-format
msgid "evaluation_status返回异常: {evaluation_status}"
msgstr ""

#: apps/log_clustering/exceptions.py:57
msgid "聚类配置不存在"
msgstr ""

#: apps/log_clustering/exceptions.py:62
#, python-brace-format
msgid "聚类配置对应告警策略错误: {index_set_id}"
msgstr ""

#: apps/log_clustering/exceptions.py:67
#, python-brace-format
msgid "聚类配置对应索引集不存在: {index_set_id}"
msgstr ""

#: apps/log_clustering/handlers/clustering_monitor.py:248
msgid "{}_日志聚类24H新类告警"
msgstr ""

#: apps/log_clustering/handlers/dataflow/constants.py:34
msgid "回流数据"
msgstr ""

#: apps/log_clustering/handlers/dataflow/constants.py:36
msgid "回流数据(tspider_storage)"
msgstr ""

#: apps/log_clustering/models.py:27
msgid "样本集ID"
msgstr ""

#: apps/log_clustering/models.py:28
msgid "样本集名称"
msgstr ""

#: apps/log_clustering/models.py:32 apps/log_clustering/models.py:37
#: apps/log_clustering/models.py:52
msgid "模型ID"
msgstr ""

#: apps/log_clustering/models.py:33
msgid "模型名称"
msgstr ""

#: apps/log_clustering/models.py:39
msgid "实验id"
msgstr ""

#: apps/log_clustering/models.py:40
msgid "实验名称"
msgstr ""

#: apps/log_clustering/models.py:41
msgid "实验状态"
msgstr ""

#: apps/log_clustering/models.py:42
msgid "最新模型实例id"
msgstr ""

#: apps/log_clustering/models.py:43
msgid "节点列表"
msgstr ""

#: apps/log_clustering/models.py:53 apps/log_clustering/models.py:89
#: apps/log_search/constants.py:47
msgid "数据指纹"
msgstr ""

#: apps/log_clustering/models.py:61 apps/log_databus/models.py:304
#: apps/log_databus/serializers.py:680
msgid "采集项id"
msgstr ""

#: apps/log_clustering/models.py:62 apps/log_databus/models.py:113
msgid "采集项英文名"
msgstr ""

#: apps/log_clustering/models.py:63 apps/log_clustering/models.py:90
#: apps/log_clustering/models.py:113 apps/log_databus/models.py:98
#: apps/log_databus/models.py:306 apps/log_databus/models.py:390
#: apps/log_search/models.py:707 apps/log_search/models.py:767
msgid "索引集id"
msgstr ""

#: apps/log_clustering/models.py:64
msgid "样本集id"
msgstr ""

#: apps/log_clustering/models.py:65
msgid "模型id"
msgstr ""

#: apps/log_clustering/models.py:66
msgid "最小日志数量"
msgstr ""

#: apps/log_clustering/models.py:67
msgid "敏感度"
msgstr ""

#: apps/log_clustering/models.py:68
msgid "预先定义的正则表达式"
msgstr ""

#: apps/log_clustering/models.py:69
msgid "分词符"
msgstr ""

#: apps/log_clustering/models.py:70
msgid "最大日志长度"
msgstr ""

#: apps/log_clustering/models.py:71
msgid "是否大小写忽略"
msgstr ""

#: apps/log_clustering/models.py:73 apps/log_databus/serializers.py:108
msgid "过滤规则"
msgstr ""

#: apps/log_clustering/models.py:75
msgid "预处理flow配置"
msgstr ""

#: apps/log_clustering/models.py:76
msgid "新类结果表id"
msgstr ""

#: apps/log_clustering/models.py:77
msgid "计算平台接入dataid"
msgstr ""

#: apps/log_clustering/models.py:78
msgid "计算平台清洗结果表"
msgstr ""

#: apps/log_clustering/models.py:79
msgid "计算平台清洗id"
msgstr ""

#: apps/log_clustering/models.py:80
msgid "入库数据源"
msgstr ""

#: apps/log_clustering/models.py:81
msgid "数据指纹开关"
msgstr ""

#: apps/log_clustering/models.py:82 apps/log_clustering/models.py:84
msgid "预处理flowid"
msgstr ""

#: apps/log_clustering/models.py:83
msgid "after_treat_flow配置"
msgstr ""

#: apps/log_clustering/models.py:85
msgid "修改after_treat_flow调用的配置"
msgstr ""

#: apps/log_clustering/models.py:91
msgid "监控策略id"
msgstr ""

#: apps/log_clustering/models.py:92 apps/log_databus/serializers.py:170
#: apps/log_extract/models.py:202 apps/log_extract/serializers.py:433
msgid "是否启用"
msgstr ""

#: apps/log_clustering/models.py:94
msgid "聚类级别"
msgstr ""

#: apps/log_clustering/models.py:96
msgid "策略类型"
msgstr ""

#: apps/log_clustering/models.py:114
msgid "通知人组id"
msgstr ""

#: apps/log_clustering/serializers.py:127
msgid "删除操作时需要提供对应strategy_id"
msgstr ""

#: apps/log_clustering/utils/monitor.py:59
msgid "{}_{}运维人员"
msgstr ""

#: apps/log_databus/constants.py:49 apps/log_search/constants.py:199
#: apps/log_search/constants.py:565
msgid "其他"
msgstr ""

#: apps/log_databus/constants.py:50
msgid "AWS"
msgstr ""

#: apps/log_databus/constants.py:51
msgid "腾讯云"
msgstr ""

#: apps/log_databus/constants.py:52
msgid "阿里云"
msgstr ""

#: apps/log_databus/constants.py:53
msgid "google"
msgstr ""

#: apps/log_databus/constants.py:54
msgid "私有自建"
msgstr ""

#: apps/log_databus/constants.py:62
#: apps/log_measure/handlers/metric_collectors/cluster.py:87
#: apps/log_measure/handlers/metrics.py:481
msgid "集群节点"
msgstr ""

#: apps/log_databus/constants.py:62
#: apps/log_measure/handlers/metric_collectors/collect.py:61
msgid "采集项容量"
msgstr ""

#: apps/log_databus/constants.py:72
msgid "未申请采集接入"
msgstr ""

#: apps/log_databus/constants.py:73
msgid "采集接入进行中"
msgstr ""

#: apps/log_databus/constants.py:74
msgid "采集接入失败"
msgstr ""

#: apps/log_databus/constants.py:75
msgid "采集接入完成"
msgstr ""

#: apps/log_databus/constants.py:123
msgid "微秒（microsecond）"
msgstr ""

#: apps/log_databus/constants.py:159 apps/log_search/constants.py:283
#: apps/log_search/constants.py:551
msgid "主机"
msgstr ""

#: apps/log_databus/constants.py:160
msgid "服务实例"
msgstr ""

#: apps/log_databus/constants.py:176
msgid "TOPO"
msgstr ""

#: apps/log_databus/constants.py:177
msgid "主机实例"
msgstr ""

#: apps/log_databus/constants.py:178
msgid "服务模板"
msgstr ""

#: apps/log_databus/constants.py:179
msgid "集群模板"
msgstr ""

#: apps/log_databus/constants.py:180
msgid "动态分组"
msgstr ""

#: apps/log_databus/constants.py:193
msgid "Transfer"
msgstr ""

#: apps/log_databus/constants.py:194
#: apps/log_search/migrations/0002_bklog_initial.py:37
#: apps/log_search/models.py:148
msgid "数据平台"
msgstr ""

#: apps/log_databus/constants.py:236
msgid "部署中"
msgstr ""

#: apps/log_databus/constants.py:237 apps/log_search/constants.py:196
#: apps/log_search/constants.py:258 apps/log_search/handlers/index_set.py:172
#: apps/log_search/models.py:568
msgid "正常"
msgstr ""

#: apps/log_databus/constants.py:238 apps/log_extract/constants.py:85
#: apps/log_trace/handlers/proto/log.py:34
msgid "失败"
msgstr ""

#: apps/log_databus/constants.py:239
msgid "部分失败"
msgstr ""

#: apps/log_databus/constants.py:240
msgid "已停用"
msgstr ""

#: apps/log_databus/constants.py:241
msgid "未知"
msgstr ""

#: apps/log_databus/constants.py:242
msgid "准备中"
msgstr ""

#: apps/log_databus/exceptions.py:32
msgid "采集配置模块异常"
msgstr ""

#: apps/log_databus/exceptions.py:37
msgid "采集配置不存在"
msgstr ""

#: apps/log_databus/exceptions.py:42
msgid "链路配置不存在"
msgstr ""

#: apps/log_databus/exceptions.py:47
msgid "参数校验异常：采集项ID不存在"
msgstr ""

#: apps/log_databus/exceptions.py:52
msgid "参数校验异常：采集任务ID不存在"
msgstr ""

#: apps/log_databus/exceptions.py:57
msgid "参数校验异常：采集实例ID不存在"
msgstr ""

#: apps/log_databus/exceptions.py:62
msgid "采集项名称已存在"
msgstr ""

#: apps/log_databus/exceptions.py:67
msgid "数据链路ID不存在"
msgstr ""

#: apps/log_databus/exceptions.py:72
msgid "无法查询到该订阅配置信息"
msgstr ""

#: apps/log_databus/exceptions.py:77
msgid "采集项在停止状态下不能编辑"
msgstr ""

#: apps/log_databus/exceptions.py:82
msgid "采集项已存在正在部署中的任务，请不要重复部署"
msgstr ""

#: apps/log_databus/exceptions.py:87
#, python-brace-format
msgid "创建或更新节点管理订阅配置异常{err}"
msgstr ""

#: apps/log_databus/exceptions.py:92
#, python-brace-format
msgid "创建数据平台异常{err}"
msgstr ""

#: apps/log_databus/exceptions.py:97
#, python-brace-format
msgid "采集项包含非该业务【{bk_biz_id}】IP，异常IP列表为: {illegal_ips}"
msgstr ""

#: apps/log_databus/exceptions.py:102
#, python-brace-format
msgid "采集项{collector_config_name_en}英文名重复"
msgstr ""

#: apps/log_databus/exceptions.py:107 apps/log_measure/exceptions.py:37
msgid "集群不存在"
msgstr ""

#: apps/log_databus/exceptions.py:112
msgid "bk_biz_id为必填字段，且为整数"
msgstr ""

#: apps/log_databus/exceptions.py:117 apps/log_esquery/exceptions.py:31
#: apps/log_search/exceptions.py:30
msgid "权限异常"
msgstr ""

#: apps/log_databus/exceptions.py:122
msgid "集群连接异常"
msgstr ""

#: apps/log_databus/exceptions.py:127
#, python-brace-format
msgid "集群连接异常, 异常信息{info}"
msgstr ""

#: apps/log_databus/exceptions.py:132 apps/log_esquery/exceptions.py:121
#, python-brace-format
msgid "未知的ES服务端版本类型\"{ip}\":\"{port}\""
msgstr ""

#: apps/log_databus/exceptions.py:137
#, python-brace-format
msgid "http探测ES版本错误, \"{msg}\""
msgstr ""

#: apps/log_databus/exceptions.py:142
msgid "集群不支持冷热数据功能"
msgstr ""

#: apps/log_databus/exceptions.py:147
msgid "集群还有未删除的采集项、第三方集群索引集"
msgstr ""

#: apps/log_databus/exceptions.py:152
#, python-brace-format
msgid "暫不支持{separator_node_action}类清洗"
msgstr ""

#: apps/log_databus/exceptions.py:157
msgid "分隔符配置解析异常"
msgstr ""

#: apps/log_databus/exceptions.py:162
msgid "字段提取预览失败，请检查提取规则与数据是否匹配"
msgstr ""

#: apps/log_databus/exceptions.py:167
msgid "解析数据时间字段异常"
msgstr ""

#: apps/log_databus/exceptions.py:172
msgid "解析时间格式异常"
msgstr ""

#: apps/log_databus/exceptions.py:177
msgid "字段名总长度超过256，暂不支持"
msgstr ""

#: apps/log_databus/exceptions.py:182
msgid "该业务已超出公共集群容量限制"
msgstr ""

#: apps/log_databus/exceptions.py:187
msgid "该数据链路名已被使用"
msgstr ""

#: apps/log_databus/exceptions.py:192
msgid "数据链路kafka、transfer集群不可修改，es集群不可删除"
msgstr ""

#: apps/log_databus/exceptions.py:197
msgid "无法匹配到行首，请确认"
msgstr ""

#: apps/log_databus/exceptions.py:202
#, python-brace-format
msgid "正则表达式不合法: \"{error}\" 请确认"
msgstr ""

#: apps/log_databus/exceptions.py:207
#, python-brace-format
msgid "kafka连接失败\"{error}\""
msgstr ""

#: apps/log_databus/exceptions.py:212
msgid "Partition信息获取失败"
msgstr ""

#: apps/log_databus/exceptions.py:217
msgid "数据链路配置获取失败"
msgstr ""

#: apps/log_databus/exceptions.py:222
msgid "采集接入ITSM-TOKEN非法"
msgstr ""

#: apps/log_databus/exceptions.py:227
msgid "已经提交采集接入申请"
msgstr ""

#: apps/log_databus/exceptions.py:232
msgid "采集接入ITSM流程服务不存在"
msgstr ""

#: apps/log_databus/exceptions.py:237
msgid "采集接入流程未完成，无法启动采集"
msgstr ""

#: apps/log_databus/exceptions.py:242
msgid "采集接入未完成"
msgstr ""

#: apps/log_databus/exceptions.py:247
msgid "后台bkdata_id_feature不存在"
msgstr ""

#: apps/log_databus/exceptions.py:252
#, python-brace-format
msgid "清洗模板{clean_template_id}不存在"
msgstr ""

#: apps/log_databus/exceptions.py:257
#, python-brace-format
msgid "该业务 {bk_biz} 已存在该模板{name}"
msgstr ""

#: apps/log_databus/exceptions.py:262
#, python-brace-format
msgid "该业务{bk_biz_id}未找到对应project"
msgstr ""

#: apps/log_databus/exceptions.py:267
msgid "归档配置不存在"
msgstr ""

#: apps/log_databus/exceptions.py:272
msgid "归档回溯不存在"
msgstr ""

#: apps/log_databus/exceptions.py:277
msgid "归档回溯已经过期"
msgstr ""

#: apps/log_databus/handlers/archive.py:147
msgid "[回溯]"
msgstr ""

#: apps/log_databus/handlers/collector.py:181
msgid "ES索引主分片大小达到{}G后分裂"
msgstr ""

#: apps/log_databus/handlers/collector.py:505
#: apps/log_databus/handlers/collector.py:1789
#: apps/log_databus/handlers/etl.py:211
msgid "[采集项]"
msgstr ""

#: apps/log_databus/handlers/collector.py:1551 apps/log_search/constants.py:222
#: apps/log_search/handlers/biz.py:206 config/default.py:628
msgid "集群"
msgstr ""

#: apps/log_databus/handlers/collector.py:1557 apps/log_search/constants.py:223
#: apps/log_search/handlers/biz.py:212
msgid "模块"
msgstr ""

#: apps/log_databus/handlers/collector_scenario/base.py:149
msgid "创建采集项时名称不能为空"
msgstr ""

#: apps/log_databus/handlers/etl_storage/base.py:137
msgid "字段不能与标准字段重复"
msgstr ""

#: apps/log_databus/handlers/etl_storage/base.py:140
msgid "字段名不符合变量规则"
msgstr ""

#: apps/log_databus/handlers/etl_storage/bk_log_delimiter.py:51
msgid "分隔符不能为空"
msgstr ""

#: apps/log_databus/handlers/etl_storage/bk_log_regexp.py:42
msgid "正则表达式不能为空"
msgstr ""

#: apps/log_databus/handlers/etl_storage/bk_log_regexp.py:47
msgid "无法匹配正则表达式"
msgstr ""

#: apps/log_databus/handlers/etl_storage/bk_log_regexp.py:76
msgid "字段未在正则表达式中定义："
msgstr ""

#: apps/log_databus/handlers/etl_storage/utils/transfer.py:66
msgid "separator_field_list类型不符合"
msgstr ""

#: apps/log_databus/handlers/storage.py:563
msgid "IP or PORT can not be reached"
msgstr ""

#: apps/log_databus/handlers/storage.py:567
#, python-format
msgid "IP or PORT can not be reached, %s"
msgstr ""

#: apps/log_databus/management/commands/sync_built_in_platform_collect.py:112
msgid "平台内置采集项 "
msgstr ""

#: apps/log_databus/models.py:71 apps/log_databus/models.py:378
msgid "采集配置ID"
msgstr ""

#: apps/log_databus/models.py:72
msgid "采集配置名称"
msgstr ""

#: apps/log_databus/models.py:73 apps/log_search/models.py:165
msgid "接入的来源APP"
msgstr ""

#: apps/log_databus/models.py:74 apps/log_search/constants.py:310
msgid "采集场景"
msgstr ""

#: apps/log_databus/models.py:76
msgid "自定义类型"
msgstr ""

#: apps/log_databus/models.py:79 apps/log_search/constants.py:309
#: apps/log_search/models.py:291
msgid "数据分类"
msgstr ""

#: apps/log_databus/models.py:81
msgid "对象类型"
msgstr ""

#: apps/log_databus/models.py:84 apps/log_databus/serializers.py:226
#: apps/log_databus/serializers.py:257
msgid "节点类型"
msgstr ""

#: apps/log_databus/models.py:86 apps/log_databus/serializers.py:293
#: apps/log_databus/serializers.py:330 apps/log_databus/serializers.py:342
msgid "采集目标"
msgstr ""

#: apps/log_databus/models.py:87
msgid "与上一次采集订阅的差异"
msgstr ""

#: apps/log_databus/models.py:88 apps/log_databus/serializers.py:478
#: apps/log_search/models.py:167
msgid "描述"
msgstr ""

#: apps/log_databus/models.py:89 apps/log_databus/models.py:261
#: apps/log_search/models.py:304
msgid "是否可用"
msgstr ""

#: apps/log_databus/models.py:90
msgid "采集链路id"
msgstr ""

#: apps/log_databus/models.py:91
msgid "采集链路data_id"
msgstr ""

#: apps/log_databus/models.py:92
msgid "采集链路data_name"
msgstr ""

#: apps/log_databus/models.py:95
msgid "节点管理订阅ID"
msgstr ""

#: apps/log_databus/models.py:96
msgid "最后一次部署任务"
msgstr ""

#: apps/log_databus/models.py:97
msgid "接入数据平台data id"
msgstr ""

#: apps/log_databus/models.py:99 apps/log_databus/serializers.py:228
#: apps/log_databus/serializers.py:259
msgid "日志字符集"
msgstr ""

#: apps/log_databus/models.py:100
msgid "params"
msgstr ""

#: apps/log_databus/models.py:101
msgid "itsm单据号"
msgstr ""

#: apps/log_databus/models.py:103
msgid "采集接入单据状态"
msgstr ""

#: apps/log_databus/models.py:105
msgid "是否能够使用独立es集群"
msgstr ""

#: apps/log_databus/models.py:106
msgid "采集打包数量"
msgstr ""

#: apps/log_databus/models.py:107
msgid "输出格式"
msgstr ""

#: apps/log_databus/models.py:108
msgid "采集器配置覆盖"
msgstr ""

#: apps/log_databus/models.py:109
msgid "ES分片数量"
msgstr ""

#: apps/log_databus/models.py:110
msgid "单shards分片大小"
msgstr ""

#: apps/log_databus/models.py:111
msgid "ES副本数"
msgstr ""

#: apps/log_databus/models.py:112
msgid "调用数据平台创建data_id失败数"
msgstr ""

#: apps/log_databus/models.py:183 apps/log_databus/models.py:184
msgid "用户采集配置"
msgstr ""

#: apps/log_databus/models.py:223
msgid "【日志采集】{}-{}-{}"
msgstr ""

#: apps/log_databus/models.py:255 apps/log_databus/serializers.py:347
msgid "链路ID"
msgstr ""

#: apps/log_databus/models.py:256 apps/log_databus/serializers.py:364
#: apps/log_databus/serializers.py:418
msgid "集群名称"
msgstr ""

#: apps/log_databus/models.py:258
msgid "kafka集群ID"
msgstr ""

#: apps/log_databus/models.py:259
msgid "transfer集群ID"
msgstr ""

#: apps/log_databus/models.py:260
msgid "es集群ID"
msgstr ""

#: apps/log_databus/models.py:262 apps/log_databus/serializers.py:171
#: apps/log_extract/models.py:104 apps/log_extract/serializers.py:108
#: apps/log_extract/serializers.py:236 apps/log_extract/serializers.py:387
msgid "备注"
msgstr ""

#: apps/log_databus/models.py:265 apps/log_databus/models.py:266
msgid "数据链路配置"
msgstr ""

#: apps/log_databus/models.py:272
msgid "容量"
msgstr ""

#: apps/log_databus/models.py:275
msgid "公共集群容量限制"
msgstr ""

#: apps/log_databus/models.py:276
msgid "存储集群容量限制"
msgstr ""

#: apps/log_databus/models.py:283
msgid "已用容量"
msgstr ""

#: apps/log_databus/models.py:286 apps/log_databus/models.py:287
msgid "业务已用容量"
msgstr ""

#: apps/log_databus/models.py:293 apps/log_trace/constants.py:92
#: apps/log_trace/handlers/trace_field_handlers.py:49
#: apps/log_trace/handlers/trace_field_handlers.py:178
msgid "状态"
msgstr ""

#: apps/log_databus/models.py:294
msgid "状态英文名"
msgstr ""

#: apps/log_databus/models.py:295
msgid "结果表id"
msgstr ""

#: apps/log_databus/models.py:296 bk_monitor/models.py:18
msgid "结果表名"
msgstr ""

#: apps/log_databus/models.py:297
msgid "结果表中文名"
msgstr ""

#: apps/log_databus/models.py:299 apps/log_search/models.py:228
#: bk_monitor/models.py:16
msgid "数据源名称"
msgstr ""

#: apps/log_databus/models.py:300
msgid "数据源中文名"
msgstr ""

#: apps/log_databus/models.py:301 apps/log_search/models.py:830
msgid "数据类型"
msgstr ""

#: apps/log_databus/models.py:302
msgid "存储类型"
msgstr ""

#: apps/log_databus/models.py:307
msgid "索引集是否被授权"
msgstr ""

#: apps/log_databus/models.py:310 apps/log_databus/models.py:311
msgid "高级清洗列表"
msgstr ""

#: apps/log_databus/models.py:316
msgid "清洗id"
msgstr ""

#: apps/log_databus/models.py:317 apps/log_search/models.py:792
msgid "模板名"
msgstr ""

#: apps/log_databus/models.py:318 apps/log_databus/models.py:331
msgid "模板类型"
msgstr ""

#: apps/log_databus/models.py:319 apps/log_databus/models.py:332
msgid "etl配置"
msgstr ""

#: apps/log_databus/models.py:320 apps/log_databus/models.py:333
msgid "etl字段"
msgstr ""

#: apps/log_databus/models.py:324 apps/log_databus/models.py:325
#: config/default.py:546
msgid "清洗模板"
msgstr ""

#: apps/log_databus/models.py:330
msgid "清洗缓存id"
msgstr ""

#: apps/log_databus/models.py:334
msgid "采集项列表"
msgstr ""

#: apps/log_databus/models.py:338 apps/log_databus/models.py:339
msgid "未完成入库暂存清洗"
msgstr ""

#: apps/log_databus/models.py:344
msgid "归档配置id"
msgstr ""

#: apps/log_databus/models.py:345
msgid "关联采集项id"
msgstr ""

#: apps/log_databus/models.py:348
msgid "快照天数"
msgstr ""

#: apps/log_databus/models.py:350
msgid "快照仓库名称"
msgstr ""

#: apps/log_databus/models.py:354 apps/log_databus/models.py:355
msgid "归档配置表"
msgstr ""

#: apps/log_databus/models.py:380
msgid "归档id"
msgstr ""

#: apps/log_databus/models.py:381
msgid "meta回溯id"
msgstr ""

#: apps/log_databus/models.py:384
msgid "到期时间"
msgstr ""

#: apps/log_databus/models.py:385
msgid "是否完成"
msgstr ""

#: apps/log_databus/models.py:386 apps/log_trace/constants.py:78
#: apps/log_trace/handlers/trace_field_handlers.py:45
msgid "耗时"
msgstr ""

#: apps/log_databus/models.py:387
msgid "存储大小"
msgstr ""

#: apps/log_databus/models.py:388
msgid "文档数量"
msgstr ""

#: apps/log_databus/models.py:389 apps/log_databus/serializers.py:693
#: apps/log_search/models.py:289
msgid "索引集名称"
msgstr ""

#: apps/log_databus/models.py:391
msgid "结果通知人"
msgstr ""

#: apps/log_databus/models.py:395 apps/log_databus/models.py:396
msgid "回溯配置表"
msgstr ""

#: apps/log_databus/serializers.py:45
msgid "管理组id列表"
msgstr ""

#: apps/log_databus/serializers.py:46
msgid "可见人员列表"
msgstr ""

#: apps/log_databus/serializers.py:54 apps/log_databus/serializers.py:64
msgid "分类标识"
msgstr ""

#: apps/log_databus/serializers.py:55 apps/log_databus/serializers.py:65
msgid "分类名称"
msgstr ""

#: apps/log_databus/serializers.py:74
msgid "服务实例id"
msgstr ""

#: apps/log_databus/serializers.py:75
msgid "节点实例id"
msgstr ""

#: apps/log_databus/serializers.py:76
msgid "节点对象"
msgstr ""

#: apps/log_databus/serializers.py:77 apps/log_databus/serializers.py:289
msgid "主机实例ip"
msgstr ""

#: apps/log_databus/serializers.py:78 apps/log_databus/serializers.py:290
msgid "蓝鲸云区域id"
msgstr ""

#: apps/log_databus/serializers.py:79 apps/log_databus/serializers.py:291
msgid "供应商id"
msgstr ""

#: apps/log_databus/serializers.py:83
msgid "匹配项所在位置"
msgstr ""

#: apps/log_databus/serializers.py:84
msgid "匹配值"
msgstr ""

#: apps/log_databus/serializers.py:85
msgid "匹配方式"
msgstr ""

#: apps/log_databus/serializers.py:86
msgid "逻辑操作符"
msgstr ""

#: apps/log_databus/serializers.py:94
msgid "过滤方式类型"
msgstr ""

#: apps/log_databus/serializers.py:96
#: apps/log_databus/views/collector_views.py:202
#: apps/log_databus/views/collector_views.py:223
#: apps/tests/log_databus/test_e2e_collectorviewsetapi.py:74
#: apps/tests/log_databus/test_e2e_collectorviewsetapi.py:95
msgid "过滤方式"
msgstr ""

#: apps/log_databus/serializers.py:103
#: apps/log_extract/components/collections/packing_component.py:51
#: apps/log_extract/models.py:79 apps/log_extract/serializers.py:343
msgid "过滤内容"
msgstr ""

#: apps/log_databus/serializers.py:106 apps/log_databus/serializers.py:451
#: apps/log_search/constants.py:453 apps/log_search/constants.py:640
msgid "分隔符"
msgstr ""

#: apps/log_databus/serializers.py:117
msgid "过滤分隔符不能为空"
msgstr ""

#: apps/log_databus/serializers.py:127
#: apps/log_databus/views/collector_views.py:195
#: apps/log_databus/views/collector_views.py:216
#: apps/tests/log_databus/test_e2e_collectorviewsetapi.py:67
#: apps/tests/log_databus/test_e2e_collectorviewsetapi.py:88
msgid "日志路径"
msgstr ""

#: apps/log_databus/serializers.py:129
msgid "行首正则"
msgstr ""

#: apps/log_databus/serializers.py:130
msgid "最多匹配行数"
msgstr ""

#: apps/log_databus/serializers.py:131
msgid "最大耗时"
msgstr ""

#: apps/log_databus/serializers.py:132
msgid "是否增量采集"
msgstr ""

#: apps/log_databus/serializers.py:133
msgid "文件扫描忽略时间"
msgstr ""

#: apps/log_databus/serializers.py:134
msgid "单行日志最大长度"
msgstr ""

#: apps/log_databus/serializers.py:136
msgid "文件扫描间隔"
msgstr ""

#: apps/log_databus/serializers.py:137
msgid "FD关联间隔"
msgstr ""

#: apps/log_databus/serializers.py:138
msgid "同时采集数"
msgstr ""

#: apps/log_databus/serializers.py:139
msgid "采集进度清理时间"
msgstr ""

#: apps/log_databus/serializers.py:142
msgid "windows事件名称"
msgstr ""

#: apps/log_databus/serializers.py:145
msgid "windows事件等级"
msgstr ""

#: apps/log_databus/serializers.py:148
msgid "windows事件ID"
msgstr ""

#: apps/log_databus/serializers.py:165
msgid "链路集群名称"
msgstr ""

#: apps/log_databus/serializers.py:166
msgid "链路允许的业务id"
msgstr ""

#: apps/log_databus/serializers.py:167
msgid "kafka集群id"
msgstr ""

#: apps/log_databus/serializers.py:168
msgid "transfer集群id"
msgstr ""

#: apps/log_databus/serializers.py:169
msgid "es集群id"
msgstr ""

#: apps/log_databus/serializers.py:175
msgid "集群种类"
msgstr ""

#: apps/log_databus/serializers.py:180 apps/log_databus/serializers.py:199
#: apps/log_databus/serializers.py:218 apps/log_databus/serializers.py:252
msgid "采集名称"
msgstr ""

#: apps/log_databus/serializers.py:182 apps/log_databus/serializers.py:220
#: apps/log_databus/serializers.py:254
msgid "采集英文名称"
msgstr ""

#: apps/log_databus/serializers.py:184 apps/log_databus/serializers.py:222
msgid "数据链路id"
msgstr ""

#: apps/log_databus/serializers.py:185 apps/log_databus/serializers.py:223
msgid "日志类型"
msgstr ""

#: apps/log_databus/serializers.py:188 apps/log_databus/serializers.py:205
#: apps/log_databus/serializers.py:530
msgid "有效时间"
msgstr ""

#: apps/log_databus/serializers.py:189 apps/log_databus/serializers.py:206
#: apps/log_databus/serializers.py:531
msgid "冷热数据生效时间"
msgstr ""

#: apps/log_databus/serializers.py:191 apps/log_databus/serializers.py:208
#: apps/log_databus/serializers.py:533
msgid "ES副本数量"
msgstr ""

#: apps/log_databus/serializers.py:194 apps/log_databus/serializers.py:202
#: apps/log_databus/serializers.py:230 apps/log_databus/serializers.py:261
msgid "备注说明"
msgstr ""

#: apps/log_databus/serializers.py:225 apps/log_databus/serializers.py:256
msgid "目标类型"
msgstr ""

#: apps/log_databus/serializers.py:227 apps/log_databus/serializers.py:258
msgid "目标节点"
msgstr ""

#: apps/log_databus/serializers.py:238 apps/log_databus/serializers.py:243
msgid "{} 该字段为必填项"
msgstr ""

#: apps/log_databus/serializers.py:297
msgid "采集项ID"
msgstr ""

#: apps/log_databus/serializers.py:301
msgid "collector_id_list不符合格式，采集项ID（多个ID用半角,分隔）"
msgstr ""

#: apps/log_databus/serializers.py:306
msgid "部署任务ID"
msgstr ""

#: apps/log_databus/serializers.py:313
msgid "task_id_list不符合格式，部署任务ID（多个ID用半角,分隔）"
msgstr ""

#: apps/log_databus/serializers.py:318 apps/log_extract/serializers.py:244
msgid "实例ID"
msgstr ""

#: apps/log_databus/serializers.py:319
#: apps/log_extract/components/collections/distribution_component.py:43
#: apps/log_extract/components/collections/packing_component.py:44
msgid "任务ID"
msgstr ""

#: apps/log_databus/serializers.py:329
msgid "数据分类名称"
msgstr ""

#: apps/log_databus/serializers.py:331
msgid "任务ID列表"
msgstr ""

#: apps/log_databus/serializers.py:332
msgid "订阅目标变更情况"
msgstr ""

#: apps/log_databus/serializers.py:333
msgid "是否可以创建基础清洗"
msgstr ""

#: apps/log_databus/serializers.py:334
msgid "数据平台索引集id列表"
msgstr ""

#: apps/log_databus/serializers.py:355
#: apps/log_extract/components/collections/distribution_component.py:45
#: apps/log_extract/components/collections/packing_component.py:49
#: version_log/models.py:42
msgid "用户名"
msgstr ""

#: apps/log_databus/serializers.py:356 apps/log_databus/serializers.py:400
msgid "密码"
msgstr ""

#: apps/log_databus/serializers.py:365 apps/log_databus/serializers.py:396
#: apps/log_databus/serializers.py:419
msgid "集群域名"
msgstr ""

#: apps/log_databus/serializers.py:366
msgid "集群端口"
msgstr ""

#: apps/log_databus/serializers.py:367 apps/log_databus/serializers.py:398
#: apps/log_databus/serializers.py:421
msgid "集群协议"
msgstr ""

#: apps/log_databus/serializers.py:368 apps/log_databus/serializers.py:403
#: apps/log_databus/serializers.py:422
msgid "凭据信息"
msgstr ""

#: apps/log_databus/serializers.py:369 apps/log_databus/serializers.py:423
msgid "是否开启冷热数据"
msgstr ""

#: apps/log_databus/serializers.py:370 apps/log_databus/serializers.py:424
msgid "热节点属性名称"
msgstr ""

#: apps/log_databus/serializers.py:371 apps/log_databus/serializers.py:425
msgid "热节点属性值"
msgstr ""

#: apps/log_databus/serializers.py:372 apps/log_databus/serializers.py:426
msgid "冷节点属性名称"
msgstr ""

#: apps/log_databus/serializers.py:373 apps/log_databus/serializers.py:427
msgid "冷节点属性值"
msgstr ""

#: apps/log_databus/serializers.py:374 apps/log_databus/serializers.py:428
msgid "ES来源类型"
msgstr ""

#: apps/log_databus/serializers.py:375 apps/log_databus/serializers.py:429
msgid "来源名称"
msgstr ""

#: apps/log_databus/serializers.py:377 apps/log_databus/serializers.py:431
msgid "可见业务范围"
msgstr ""

#: apps/log_databus/serializers.py:386 apps/log_databus/serializers.py:440
msgid "当冷热数据处于开启状态时，冷热节点属性配置不能为空"
msgstr ""

#: apps/log_databus/serializers.py:389 apps/log_databus/serializers.py:444
msgid "当来源类型为其他时，需要传入来源名称"
msgstr ""

#: apps/log_databus/serializers.py:397 apps/log_databus/serializers.py:420
msgid "端口"
msgstr ""

#: apps/log_databus/serializers.py:399
msgid "用户"
msgstr ""

#: apps/log_databus/serializers.py:401
msgid "是否包含集群信息"
msgstr ""

#: apps/log_databus/serializers.py:402
msgid "是否使用默认用户信息"
msgstr ""

#: apps/log_databus/serializers.py:449 apps/log_search/constants.py:454
msgid "正则表达式"
msgstr ""

#: apps/log_databus/serializers.py:453
msgid "是否保留原文"
msgstr ""

#: apps/log_databus/serializers.py:457 apps/log_databus/serializers.py:526
#: apps/log_databus/serializers.py:646 apps/log_databus/serializers.py:653
msgid "清洗类型"
msgstr ""

#: apps/log_databus/serializers.py:459
msgid "日志内容"
msgstr ""

#: apps/log_databus/serializers.py:463
msgid "日志样例"
msgstr ""

#: apps/log_databus/serializers.py:464
msgid "行首正则表达式"
msgstr ""

#: apps/log_databus/serializers.py:468 apps/log_search/constants.py:316
msgid "时间格式"
msgstr ""

#: apps/log_databus/serializers.py:469 apps/log_search/models.py:166
msgid "时区"
msgstr ""

#: apps/log_databus/serializers.py:470
msgid "时间内容"
msgstr ""

#: apps/log_databus/serializers.py:474
msgid "字段顺序"
msgstr ""

#: apps/log_databus/serializers.py:475
msgid "字段名称"
msgstr ""

#: apps/log_databus/serializers.py:476
msgid "别名"
msgstr ""

#: apps/log_databus/serializers.py:477 apps/log_trace/handlers/proto/otlp.py:73
#: bk_monitor/models.py:21
msgid "类型"
msgstr ""

#: apps/log_databus/serializers.py:479
msgid "是否分词"
msgstr ""

#: apps/log_databus/serializers.py:480
msgid "是否维度"
msgstr ""

#: apps/log_databus/serializers.py:481
msgid "是否时间字段"
msgstr ""

#: apps/log_databus/serializers.py:482 apps/models.py:265
msgid "是否删除"
msgstr ""

#: apps/log_databus/serializers.py:483
msgid "是否内置字段"
msgstr ""

#: apps/log_databus/serializers.py:484 apps/log_databus/serializers.py:528
#: apps/log_databus/serializers.py:648 apps/log_databus/serializers.py:655
#: apps/log_search/models.py:594
msgid "字段配置"
msgstr ""

#: apps/log_databus/serializers.py:490
msgid "参数不能为空：field_name/field_type"
msgstr ""

#: apps/log_databus/serializers.py:495
msgid "字段别名不能与标准字段重复"
msgstr ""

#: apps/log_databus/serializers.py:497
msgid "字段名称不能与标准字段重复"
msgstr ""

#: apps/log_databus/serializers.py:508
msgid "只有字符串类型的字段才可以设置为分词"
msgstr ""

#: apps/log_databus/serializers.py:513
msgid "时间字段需配置时区、格式"
msgstr ""

#: apps/log_databus/serializers.py:517 apps/log_databus/serializers.py:520
msgid "时间字段类型与格式不匹配"
msgstr ""

#: apps/log_databus/serializers.py:535 apps/log_search/models.py:299
msgid "查看权限"
msgstr ""

#: apps/log_databus/serializers.py:542
msgid "[字段提取]请输入需要提取的字段信息"
msgstr ""

#: apps/log_databus/serializers.py:546
msgid "存储索引名不能包含bklog关键字"
msgstr ""

#: apps/log_databus/serializers.py:561
msgid "分隔符必须指定field_index"
msgstr ""

#: apps/log_databus/serializers.py:570
msgid "仅可以设置一个时间字段"
msgstr ""

#: apps/log_databus/serializers.py:573
msgid "清洗需要配置有效字段"
msgstr ""

#: apps/log_databus/serializers.py:581
msgid "itsm单号"
msgstr ""

#: apps/log_databus/serializers.py:582
msgid "itsm单据标题"
msgstr ""

#: apps/log_databus/serializers.py:583
msgid "itsm单据地址"
msgstr ""

#: apps/log_databus/serializers.py:584
msgid "itsm单据状态"
msgstr ""

#: apps/log_databus/serializers.py:585
msgid "itsm最近更新者"
msgstr ""

#: apps/log_databus/serializers.py:586
msgid "itsm最近更新时间"
msgstr ""

#: apps/log_databus/serializers.py:587
msgid "itsm审批结果"
msgstr ""

#: apps/log_databus/serializers.py:588
msgid "itsmtoken"
msgstr ""

#: apps/log_databus/serializers.py:592
msgid "期待接入日期"
msgstr ""

#: apps/log_databus/serializers.py:593
msgid "单条日志大小(bytes)"
msgstr ""

#: apps/log_databus/serializers.py:594
msgid "单机流量峰值（kB/S）"
msgstr ""

#: apps/log_databus/serializers.py:595
msgid "单机增长日志量"
msgstr ""

#: apps/log_databus/serializers.py:596
msgid "预计接入主机数"
msgstr ""

#: apps/log_databus/serializers.py:597
msgid "日志保留天数"
msgstr ""

#: apps/log_databus/serializers.py:598
msgid "热数据天数"
msgstr ""

#: apps/log_databus/serializers.py:599
msgid "申请原因"
msgstr ""

#: apps/log_databus/serializers.py:603
msgid "内网ip"
msgstr ""

#: apps/log_databus/serializers.py:604
msgid "云区域Id"
msgstr ""

#: apps/log_databus/serializers.py:605
msgid "主机id"
msgstr ""

#: apps/log_databus/serializers.py:611
msgid "检索关键词"
msgstr ""

#: apps/log_databus/serializers.py:612
msgid "清洗配置类型"
msgstr ""

#: apps/log_databus/serializers.py:613 apps/log_databus/serializers.py:624
#: apps/log_search/serializers.py:45 apps/log_search/serializers.py:274
msgid "页码"
msgstr ""

#: apps/log_databus/serializers.py:614 apps/log_databus/serializers.py:625
#: apps/log_search/serializers.py:275
msgid "页面大小"
msgstr ""

#: apps/log_databus/serializers.py:619 apps/log_databus/serializers.py:630
msgid "分页参数不能为负数"
msgstr ""

#: apps/log_databus/serializers.py:641
msgid "是否是轮询请求"
msgstr ""

#: apps/log_databus/serializers.py:645
msgid "清洗模板名"
msgstr ""

#: apps/log_databus/serializers.py:675 apps/log_databus/serializers.py:710
msgid "分页page"
msgstr ""

#: apps/log_databus/serializers.py:676 apps/log_databus/serializers.py:711
msgid "分页pagesize"
msgstr ""

#: apps/log_databus/serializers.py:681
msgid "目标es集群快照仓库"
msgstr ""

#: apps/log_databus/serializers.py:682 apps/log_databus/serializers.py:687
msgid "快照存储时间配置"
msgstr ""

#: apps/log_databus/serializers.py:694
msgid "数据开始时间"
msgstr ""

#: apps/log_databus/serializers.py:695
msgid "数据结束时间"
msgstr ""

#: apps/log_databus/serializers.py:696 apps/log_databus/serializers.py:701
msgid "指定过期时间"
msgstr ""

#: apps/log_databus/serializers.py:697
msgid "通知人"
msgstr ""

#: apps/log_databus/serializers.py:705
msgid "回溯id"
msgstr ""

#: apps/log_databus/serializers.py:719
msgid "归档回溯配置list"
msgstr ""

#: apps/log_databus/views/collector_views.py:189
#: apps/log_search/constants.py:460
#: apps/tests/log_databus/test_e2e_collectorviewsetapi.py:61
msgid "行日志"
msgstr ""

#: apps/log_databus/views/collector_views.py:210
#: apps/log_search/constants.py:461
#: apps/tests/log_databus/test_e2e_collectorviewsetapi.py:82
msgid "段日志"
msgstr ""

#: apps/log_esquery/esquery/client/QueryClientEs.py:121
#: apps/log_esquery/esquery/client/QueryClientLog.py:134
#: apps/log_search/exceptions.py:68
msgid "EsClient链接失败"
msgstr ""

#: apps/log_esquery/esquery/client/QueryClientLog.py:305
msgid "结果表不存在"
msgstr ""

#: apps/log_esquery/exceptions.py:36
#, python-format, python-brace-format
msgid "查询的接入类型\"{scenario_id}\"不在接入范围%s"
msgstr ""

#: apps/log_esquery/exceptions.py:41
msgid "请输入索引集ID"
msgstr ""

#: apps/log_esquery/exceptions.py:46
msgid "此接口暂未对外，请确认"
msgstr ""

#: apps/log_esquery/exceptions.py:56
msgid "场景类型异常"
msgstr ""

#: apps/log_esquery/exceptions.py:61
#, python-brace-format
msgid ""
"该索引\"{result_table_id}\"不存在，所在的scenario_id不存在\"{scenario_id}\""
msgstr ""

#: apps/log_esquery/exceptions.py:66
#, python-brace-format
msgid "该索引所在的scenario_id不唯一\"{scenario_id_list}\""
msgstr ""

#: apps/log_esquery/exceptions.py:71
msgid "scenario空异常"
msgstr ""

#: apps/log_esquery/exceptions.py:76
#, python-brace-format
msgid "该索引所在的source_id不唯一\"{source_id_list}\""
msgstr ""

#: apps/log_esquery/exceptions.py:81
msgid ""
"存储集群ID(storage_cluster_id)信息空异常，当查询场景（scenario_id）不是bkdata"
"时候需要传入正确的storage_cluster_id"
msgstr ""

#: apps/log_esquery/exceptions.py:86
#, python-brace-format
msgid "该索引\"{result_table_id}\"不存在，所在的Source_id不存在\"{source_id}\""
msgstr ""

#: apps/log_esquery/exceptions.py:91
#, python-brace-format
msgid ""
"该索引\"{result_table_id}\"所在查询场景\"{scenario_id}\"与查询连接信息所需场"
"景不匹配"
msgstr ""

#: apps/log_esquery/exceptions.py:101
msgid "时间异常"
msgstr ""

#: apps/log_esquery/exceptions.py:106
msgid "时间字段空异常"
msgstr ""

#: apps/log_esquery/exceptions.py:116 apps/log_esquery/exceptions.py:131
msgid "客户端版本异常"
msgstr ""

#: apps/log_esquery/exceptions.py:136
msgid "缺少连接ES的账号密码"
msgstr ""

#: apps/log_esquery/exceptions.py:141
#, python-brace-format
msgid "连接ES错误\"{error}\""
msgstr ""

#: apps/log_esquery/exceptions.py:146
#, python-brace-format
msgid "加载EsClient客户端失败，场景\"{scenario_id}\"，版本\"{version}"
msgstr ""

#: apps/log_esquery/exceptions.py:151
#, python-brace-format
msgid "EsClient查询错误\"{error}\""
msgstr ""

#: apps/log_esquery/exceptions.py:156
#, python-brace-format
msgid "原生ES主机\"{host}\"和端口\"{port}错误\""
msgstr ""

#: apps/log_esquery/exceptions.py:161
#, python-brace-format
msgid "从meta获取连接信息失败{message}"
msgstr ""

#: apps/log_esquery/exceptions.py:166
#, python-brace-format
msgid "EsClient scroll查询错误\"{error}\""
msgstr ""

#: apps/log_esquery/exceptions.py:171
#, python-brace-format
msgid "EsClient Alias查询错误\"{error}\""
msgstr ""

#: apps/log_esquery/exceptions.py:176
#, python-brace-format
msgid "EsClient cat indices查询错误\"{error}\""
msgstr ""

#: apps/log_esquery/exceptions.py:181
msgid "EsClient 查询错误"
msgstr ""

#: apps/log_esquery/exceptions.py:186
msgid "Es 查询超时"
msgstr ""

#: apps/log_esquery/exceptions.py:196 apps/log_trace/exceptions.py:30
msgid "查询异常"
msgstr ""

#: apps/log_esquery/exceptions.py:201 apps/log_search/exceptions.py:291
msgid "构建查询参数失败"
msgstr ""

#: apps/log_esquery/exceptions.py:206
#, python-brace-format
msgid "拉取ES字段失败, ERROR: \"{error}\""
msgstr ""

#: apps/log_esquery/exceptions.py:211
#, python-brace-format
msgid "查询上下文场景错误, ERROR Scenario_id is : \"{scenario_id}\""
msgstr ""

#: apps/log_esquery/exceptions.py:216
#, python-brace-format
msgid "查询DSL错误, ERROR DSL is : \"{dsl}\""
msgstr ""

#: apps/log_esquery/exceptions.py:221 apps/log_search/exceptions.py:286
#, python-brace-format
msgid "索引集【{index_set_id}】没有已审批通过的索引信息"
msgstr ""

#: apps/log_esquery/exceptions.py:226 apps/log_search/exceptions.py:236
#, python-brace-format
msgid "找不到索引集\"{index_set_id}\"的相关信息"
msgstr ""

#: apps/log_esquery/exceptions.py:231
#, python-brace-format
msgid "找不到索引集\"{index_set_id}\"的相关信息time_field信息"
msgstr ""

#: apps/log_esquery/serializers.py:54
msgid "默认使用time_range的方式检索"
msgstr ""

#: apps/log_esquery/serializers.py:56
msgid "是否包含开始时间点(gte/gt)"
msgstr ""

#: apps/log_esquery/serializers.py:57
msgid "是否包含结束时间点(lte/lt)"
msgstr ""

#: apps/log_esquery/serializers.py:116 apps/log_esquery/serializers.py:319
#: apps/log_esquery/serializers.py:358
msgid "第三方ES需传集群ID：storage_cluster_id"
msgstr ""

#: apps/log_esquery/serializers.py:122
msgid "请提供时间字段"
msgstr ""

#: apps/log_esquery/serializers.py:189 apps/log_esquery/serializers.py:227
#: apps/log_esquery/serializers.py:246 apps/log_esquery/serializers.py:298
msgid "indices该字段是必填项"
msgstr ""

#: apps/log_esquery/serializers.py:208
msgid "bk_biz_id该字段是必填项"
msgstr ""

#: apps/log_esquery/serializers.py:210 apps/log_esquery/serializers.py:229
#: apps/log_esquery/serializers.py:248 apps/log_esquery/serializers.py:280
#: apps/log_esquery/serializers.py:300
msgid "storage_cluster_id该字段是必填项"
msgstr ""

#: apps/log_esquery/serializers.py:253
msgid "转发的url"
msgstr ""

#: apps/log_esquery/serializers.py:264
msgid "非法的url路径"
msgstr ""

#: apps/log_esquery/serializers.py:282
msgid "indices 该字段是必填项"
msgstr ""

#: apps/log_extract/admin.py:70 apps/log_extract/models.py:191
#: apps/log_extract/serializers.py:409 apps/log_extract/serializers.py:426
msgid "链路类型"
msgstr ""

#: apps/log_extract/components/collections/cos_upload_component.py:35
msgid "Cos 文件上传"
msgstr ""

#: apps/log_extract/components/collections/cos_upload_component.py:41
msgid "job instance id"
msgstr ""

#: apps/log_extract/components/collections/cos_upload_component.py:42
msgid "pack file name"
msgstr ""

#: apps/log_extract/components/collections/cos_upload_component.py:92
#: apps/log_extract/tasks/extract.py:362
msgid "Cos Upload 正在执行中"
msgstr ""

#: apps/log_extract/components/collections/cos_upload_component.py:97
#: apps/log_extract/tasks/extract.py:367
msgid "上传网盘异常: {}"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:37
msgid "文件分发"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:44
msgid "源文件信息列表"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:46
#: apps/log_extract/components/collections/packing_component.py:53
msgid "执行脚本机器的用户名"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:48
#: apps/log_extract/components/collections/packing_component.py:48
#: apps/log_extract/models.py:52 apps/log_extract/serializers.py:285
msgid "作业执行人"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:53
msgid "分发任务instance id"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:54
msgid "分发服务器IP地址"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:55
msgid "分发服务器中转文件路径"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:56
msgid "分发后打包步骤的打包路径"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:78
#: apps/log_extract/tasks/extract.py:254
msgid "请配置链路中转服务器"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:133
#: apps/log_extract/components/collections/packing_component.py:152
#: apps/log_extract/tasks/extract.py:202 apps/log_extract/tasks/extract.py:308
msgid "脚本正在执行中"
msgstr ""

#: apps/log_extract/components/collections/distribution_component.py:139
#: apps/log_extract/tasks/extract.py:314
msgid "文件分发异常({})"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:38
msgid "文件打包"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:45
#: apps/log_extract/models.py:76 apps/log_extract/serializers.py:330
#: apps/log_extract/serializers.py:340
msgid "文件列表"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:46
msgid "目标ip列表"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:50
#: apps/log_extract/models.py:78 apps/log_extract/serializers.py:106
#: apps/log_extract/serializers.py:342
msgid "过滤类型"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:52
msgid "打包路径"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:58
msgid "任务实例ID"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:59
msgid "task输出内容"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:60
msgid "打包后的文件路径"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:61
msgid "上传的服务器IP"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:162
#: apps/log_extract/tasks/extract.py:212
msgid "目标机器没有足够的储存"
msgstr ""

#: apps/log_extract/components/collections/packing_component.py:167
#: apps/log_extract/tasks/extract.py:217
msgid "任务打包异常: {}"
msgstr ""

#: apps/log_extract/constants.py:45
msgid "初始化"
msgstr ""

#: apps/log_extract/constants.py:46
msgid "初始化任务调度"
msgstr ""

#: apps/log_extract/constants.py:47
msgid "文件打包中"
msgstr ""

#: apps/log_extract/constants.py:48
msgid "文件分发中"
msgstr ""

#: apps/log_extract/constants.py:49
msgid "分发文件打包中"
msgstr ""

#: apps/log_extract/constants.py:50
msgid "文件上传中"
msgstr ""

#: apps/log_extract/constants.py:51 apps/log_extract/handlers/tasks.py:233
msgid "分发到网盘中"
msgstr ""

#: apps/log_extract/constants.py:52
msgid "已完成"
msgstr ""

#: apps/log_extract/constants.py:53
msgid "已过期"
msgstr ""

#: apps/log_extract/constants.py:54
msgid "文件提取异常"
msgstr ""

#: apps/log_extract/constants.py:68
msgid "按关键字过滤行"
msgstr ""

#: apps/log_extract/constants.py:69
msgid "按行数过滤"
msgstr ""

#: apps/log_extract/constants.py:70
msgid "获取最新n行"
msgstr ""

#: apps/log_extract/constants.py:71
msgid "按关键字范围过滤行"
msgstr ""

#: apps/log_extract/constants.py:83
msgid "未执行"
msgstr ""

#: apps/log_extract/constants.py:84
msgid "执行中"
msgstr ""

#: apps/log_extract/constants.py:86
msgid "暂停"
msgstr ""

#: apps/log_extract/constants.py:87
msgid "已经终止"
msgstr ""

#: apps/log_extract/constants.py:88
msgid "已经结束"
msgstr ""

#: apps/log_extract/constants.py:102
msgid "与"
msgstr ""

#: apps/log_extract/constants.py:103
msgid "或"
msgstr ""

#: apps/log_extract/constants.py:104
msgid "非"
msgstr ""

#: apps/log_extract/constants.py:115
msgid "按topo选择目标"
msgstr ""

#: apps/log_extract/constants.py:115
msgid "按module选择目标"
msgstr ""

#: apps/log_extract/constants.py:125 apps/log_extract/constants.py:129
msgid "内网链路"
msgstr ""

#: apps/log_extract/constants.py:125
msgid "腾讯云cos链路"
msgstr ""

#: apps/log_extract/constants.py:125 apps/log_extract/constants.py:129
msgid "bk repo链路"
msgstr ""

#: apps/log_extract/constants.py:138
msgid "近一天"
msgstr ""

#: apps/log_extract/constants.py:138
msgid "近一周"
msgstr ""

#: apps/log_extract/constants.py:138
msgid "近一月"
msgstr ""

#: apps/log_extract/constants.py:138
msgid "所有"
msgstr ""

#: apps/log_extract/constants.py:138 apps/log_search/constants.py:430
msgid "自定义"
msgstr ""

#: apps/log_extract/exceptions.py:27
msgid "日志提取模块异常"
msgstr ""

#: apps/log_extract/exceptions.py:32
msgid "请配置云石账密信息"
msgstr ""

#: apps/log_extract/exceptions.py:37
msgid "请配置中转服务器信息"
msgstr ""

#: apps/log_extract/exceptions.py:42
msgid "该Task ID 不存在"
msgstr ""

#: apps/log_extract/exceptions.py:47 apps/log_extract/exceptions.py:167
#, python-brace-format
msgid "拉取topo结构失败：{error}, 请求参数：{params} "
msgstr ""

#: apps/log_extract/exceptions.py:52
#, python-brace-format
msgid "获取全部topo为空,请求参数：{params} "
msgstr ""

#: apps/log_extract/exceptions.py:57
msgid "获取任务详情失败，您不是该任务的创建者"
msgstr ""

#: apps/log_extract/exceptions.py:62
#, python-brace-format
msgid ""
"选择文件不可下载，请确认所配置策略，不可下载文件：{failed_download_file_list}"
msgstr ""

#: apps/log_extract/exceptions.py:67
msgid "重新创建任务失败，您不是原任务的创建者"
msgstr ""

#: apps/log_extract/exceptions.py:72
msgid "禁止删除任务！"
msgstr ""

#: apps/log_extract/exceptions.py:77 apps/log_extract/exceptions.py:112
#, python-brace-format
msgid "{message}"
msgstr ""

#: apps/log_extract/exceptions.py:82
msgid "当前任务已过期"
msgstr ""

#: apps/log_extract/exceptions.py:87
msgid "当前任务已达最大下载次数"
msgstr ""

#: apps/log_extract/exceptions.py:92
msgid "创建任务异常"
msgstr ""

#: apps/log_extract/exceptions.py:97
msgid "当前任务不可下载"
msgstr ""

#: apps/log_extract/exceptions.py:102
msgid "下载任务失败，您没有此任务的下载权限"
msgstr ""

#: apps/log_extract/exceptions.py:107
msgid "更新任务信息失败，你不是此任务的创建者"
msgstr ""

#: apps/log_extract/exceptions.py:117
msgid "请管理员配置提取链路"
msgstr ""

#: apps/log_extract/exceptions.py:122
msgid "提取链路不存在，请管理员检查"
msgstr ""

#: apps/log_extract/exceptions.py:127
msgid "下载目标文件不存在"
msgstr ""

#: apps/log_extract/exceptions.py:132
#, python-brace-format
msgid "访问{request_dir}目录错误，请检查是否已授权"
msgstr ""

#: apps/log_extract/exceptions.py:137
msgid "用户获取可访问目录失败，请重新选择服务器"
msgstr ""

#: apps/log_extract/exceptions.py:142
#, python-brace-format
msgid "未对用户授权模块{request_module}"
msgstr ""

#: apps/log_extract/exceptions.py:147
msgid "文件或目录不存在"
msgstr ""

#: apps/log_extract/exceptions.py:152
msgid "所选服务器器中同时包含windows与linux，请重新选择服务器"
msgstr ""

#: apps/log_extract/exceptions.py:157
#, python-brace-format
msgid "存在{mismatch_number}个服务器无对应TOPO，请重新选择服务器"
msgstr ""

#: apps/log_extract/exceptions.py:162
msgid "所选择的服务器授权目录或文件后缀无交集，请重新选择服务器"
msgstr ""

#: apps/log_extract/exceptions.py:172
#, python-brace-format
msgid "获取topo为空,请求参数：{params} "
msgstr ""

#: apps/log_extract/exceptions.py:177
msgid "暂无权限访问此页面,请联系管理员"
msgstr ""

#: apps/log_extract/exceptions.py:182
msgid "过滤topo为空,请检查策略配置"
msgstr ""

#: apps/log_extract/exceptions.py:187
msgid "预览文件超时，请重试"
msgstr ""

#: apps/log_extract/exceptions.py:192
msgid "文件预览异常"
msgstr ""

#: apps/log_extract/exceptions.py:197
#, python-brace-format
msgid "ID为{strategy_id}的策略不存在!"
msgstr ""

#: apps/log_extract/exceptions.py:202
msgid "参数异常：策略ID不存在"
msgstr ""

#: apps/log_extract/exceptions.py:207
msgid "该业务下已存在相同的策略名"
msgstr ""

#: apps/log_extract/exceptions.py:212
msgid "您在该业务下尚未拥有下载策略，联系业务运维进行下载策略授权"
msgstr ""

#: apps/log_extract/exceptions.py:217
msgid "您在该业务下尚未拥有下载策略，请先通过'管理-日志提取配置'添加策略"
msgstr ""

#: apps/log_extract/exceptions.py:222
msgid "策略执行人配置不合法"
msgstr ""

#: apps/log_extract/exceptions.py:227
msgid "该用户组不存在"
msgstr ""

#: apps/log_extract/exceptions.py:232
#, python-brace-format
msgid "搜索文件失败,失败原因: {message}"
msgstr ""

#: apps/log_extract/exceptions.py:237
#, python-brace-format
msgid " {message}"
msgstr ""

#: apps/log_extract/exceptions.py:243
#, python-brace-format
msgid ""
"[periodic_clear_timeout_pipeline_task]撤销超时pipeline任务失败：{exceptions} "
"task_id=>{task_id}, pipeline_id=>{pipeline_id}"
msgstr ""

#: apps/log_extract/exceptions.py:250
msgid "查找的日志提取链路不存在"
msgstr ""

#: apps/log_extract/exceptions.py:255
msgid "查找的日志提取链路type不支持"
msgstr ""

#: apps/log_extract/exceptions.py:260
msgid "不能修改或者删除还在进行中任务的链路"
msgstr ""

#: apps/log_extract/exceptions.py:265
msgid "同名日志提取链路已存在"
msgstr ""

#: apps/log_extract/fileserver.py:166
msgid "获取脚本内容异常({})"
msgstr ""

#: apps/log_extract/handlers/explorer.py:89
#, python-brace-format
msgid ""
"{strategy_name} 策略中的执行人 {operator} 没有JOB的操作权限,无法执行下载操作"
msgstr ""

#: apps/log_extract/handlers/explorer.py:128
msgid "文件预览异常({})"
msgstr ""

#: apps/log_extract/models.py:44
msgid "策略ID"
msgstr ""

#: apps/log_extract/models.py:46 apps/log_extract/serializers.py:278
msgid "策略名称"
msgstr ""

#: apps/log_extract/models.py:47 apps/log_extract/serializers.py:251
msgid "用户ID"
msgstr ""

#: apps/log_extract/models.py:48 apps/log_extract/serializers.py:252
#: apps/log_extract/serializers.py:281
msgid "目标选择类型"
msgstr ""

#: apps/log_extract/models.py:49 apps/log_extract/serializers.py:253
msgid "模块列表"
msgstr ""

#: apps/log_extract/models.py:50 apps/log_extract/serializers.py:254
msgid "目录列表"
msgstr ""

#: apps/log_extract/models.py:51 apps/log_extract/serializers.py:255
msgid "文件类型"
msgstr ""

#: apps/log_extract/models.py:73
msgid "任务记录id"
msgstr ""

#: apps/log_extract/models.py:75 apps/log_extract/serializers.py:55
#: apps/log_extract/serializers.py:329 apps/log_extract/serializers.py:339
msgid "业务机器ip"
msgstr ""

#: apps/log_extract/models.py:81
msgid "当前文件下载状态"
msgstr ""

#: apps/log_extract/models.py:82
msgid "任务过期时间"
msgstr ""

#: apps/log_extract/models.py:83
msgid "流水线ID"
msgstr ""

#: apps/log_extract/models.py:84
msgid "流水线组件ID"
msgstr ""

#: apps/log_extract/models.py:86
msgid "文件分发ID"
msgstr ""

#: apps/log_extract/models.py:89
msgid "上传票据"
msgstr ""

#: apps/log_extract/models.py:90
msgid "上传随机值"
msgstr ""

#: apps/log_extract/models.py:93
msgid "任务上传ID"
msgstr ""

#: apps/log_extract/models.py:94
msgid "云石上传ID"
msgstr ""

#: apps/log_extract/models.py:97
msgid "云石文件路径"
msgstr ""

#: apps/log_extract/models.py:99
msgid "云石任务ID"
msgstr ""

#: apps/log_extract/models.py:100
msgid "云石下载业务ID"
msgstr ""

#: apps/log_extract/models.py:101
msgid "下载票据"
msgstr ""

#: apps/log_extract/models.py:102
msgid "下载随机值"
msgstr ""

#: apps/log_extract/models.py:103 apps/log_extract/serializers.py:235
#: apps/log_extract/serializers.py:344
msgid "任务过程信息"
msgstr ""

#: apps/log_extract/models.py:106 apps/log_extract/serializers.py:109
msgid "预览目录"
msgstr ""

#: apps/log_extract/models.py:107 apps/log_extract/serializers.py:110
msgid "预览地址ip"
msgstr ""

#: apps/log_extract/models.py:108 apps/log_extract/serializers.py:111
msgid "预览日期"
msgstr ""

#: apps/log_extract/models.py:109 apps/log_extract/serializers.py:114
msgid "预览是否搜索子目录"
msgstr ""

#: apps/log_extract/models.py:110
msgid "预览开始日期"
msgstr ""

#: apps/log_extract/models.py:111
msgid "预览结束日期"
msgstr ""

#: apps/log_extract/models.py:113
msgid "额外数据"
msgstr ""

#: apps/log_extract/models.py:114
msgid "cos对象文件名称"
msgstr ""

#: apps/log_extract/models.py:115 apps/log_extract/models.py:190
#: apps/log_extract/serializers.py:408
msgid "链路id"
msgstr ""

#: apps/log_extract/models.py:166
msgid "总耗时"
msgstr ""

#: apps/log_extract/models.py:171
msgid "IP数量"
msgstr ""

#: apps/log_extract/models.py:185
msgid "下载文件统计"
msgstr ""

#: apps/log_extract/models.py:189 apps/log_extract/serializers.py:407
#: apps/log_extract/serializers.py:425
msgid "链路名称"
msgstr ""

#: apps/log_extract/models.py:192 apps/log_extract/serializers.py:427
msgid "执行人"
msgstr ""

#: apps/log_extract/models.py:193 apps/log_extract/serializers.py:428
msgid "执行bk_biz_id"
msgstr ""

#: apps/log_extract/models.py:194 apps/log_extract/serializers.py:429
msgid "腾讯云SecretId"
msgstr ""

#: apps/log_extract/models.py:194 apps/log_extract/models.py:195
#: apps/log_extract/models.py:197 apps/log_extract/models.py:200
msgid "内网链路不需要填写"
msgstr ""

#: apps/log_extract/models.py:195 apps/log_extract/serializers.py:430
msgid "腾讯云SecretKey"
msgstr ""

#: apps/log_extract/models.py:197 apps/log_extract/serializers.py:431
msgid "腾讯云Cos桶名称"
msgstr ""

#: apps/log_extract/models.py:200 apps/log_extract/serializers.py:432
msgid "腾讯云Cos区域"
msgstr ""

#: apps/log_extract/models.py:203 apps/log_extract/serializers.py:411
#: apps/models.py:204
msgid "创建时间"
msgstr ""

#: apps/log_extract/models.py:206 apps/log_extract/models.py:207
msgid ""
"提取链路 (第一次配置链路之后 需要重新部署saas && 暂时只支持linux及安装了cgwin"
"的系统)"
msgstr ""

#: apps/log_extract/models.py:211 apps/log_extract/serializers.py:419
msgid "挂载目录"
msgstr ""

#: apps/log_extract/models.py:212 apps/log_extract/serializers.py:420
msgid "主机云区域id"
msgstr ""

#: apps/log_extract/models.py:213 apps/log_extract/serializers.py:421
msgid "主机ip"
msgstr ""

#: apps/log_extract/models.py:217 apps/log_extract/models.py:218
msgid "链路中转机"
msgstr ""

#: apps/log_extract/serializers.py:56
msgid "业务机器云区域id"
msgstr ""

#: apps/log_extract/serializers.py:61 apps/log_extract/serializers.py:94
msgid "业务机器列表"
msgstr ""

#: apps/log_extract/serializers.py:62
msgid "文件路径"
msgstr ""

#: apps/log_extract/serializers.py:63
msgid "是否搜索子目录"
msgstr ""

#: apps/log_extract/serializers.py:64
msgid "时间跨度"
msgstr ""

#: apps/log_extract/serializers.py:71 apps/log_extract/serializers.py:140
#: apps/log_extract/serializers.py:307
msgid "请指定正确的目录"
msgstr ""

#: apps/log_extract/serializers.py:73
msgid "请指定正确的时间跨度"
msgstr ""

#: apps/log_extract/serializers.py:84
msgid "时间格式不正确"
msgstr ""

#: apps/log_extract/serializers.py:86
msgid "请指定开始和结束时间"
msgstr ""

#: apps/log_extract/serializers.py:99 apps/log_search/serializers.py:250
msgid "搜索关键字"
msgstr ""

#: apps/log_extract/serializers.py:104
msgid "业务机器IP"
msgstr ""

#: apps/log_extract/serializers.py:105
msgid "目标文件路径"
msgstr ""

#: apps/log_extract/serializers.py:107
msgid "过滤参数"
msgstr ""

#: apps/log_extract/serializers.py:112
msgid "预览开始时间"
msgstr ""

#: apps/log_extract/serializers.py:113
msgid "预览结束时间"
msgstr ""

#: apps/log_extract/serializers.py:115
msgid "提取链路id"
msgstr ""

#: apps/log_extract/serializers.py:122
msgid "请指定正确的过滤类型"
msgstr ""

#: apps/log_extract/serializers.py:131
msgid "提取链路不存在"
msgstr ""

#: apps/log_extract/serializers.py:135
msgid "同时下载的文件数不能超过{}"
msgstr ""

#: apps/log_extract/serializers.py:145
msgid "请指定正确的预览日期模式"
msgstr ""

#: apps/log_extract/serializers.py:157 apps/log_extract/serializers.py:203
msgid "过滤关键词必须为字符串类型"
msgstr ""

#: apps/log_extract/serializers.py:159
msgid "过滤关键词不可为空"
msgstr ""

#: apps/log_extract/serializers.py:161 apps/log_extract/serializers.py:207
msgid "过滤关键字超过长度"
msgstr ""

#: apps/log_extract/serializers.py:164
msgid "过滤关键词请勿包含空格"
msgstr ""

#: apps/log_extract/serializers.py:169
msgid "过滤关键字类型不正确"
msgstr ""

#: apps/log_extract/serializers.py:177
msgid "行数必须为整型"
msgstr ""

#: apps/log_extract/serializers.py:180 apps/log_extract/serializers.py:192
msgid "行数不可为空"
msgstr ""

#: apps/log_extract/serializers.py:183 apps/log_extract/serializers.py:186
msgid "请确认起始行数与终止行数的大小关系"
msgstr ""

#: apps/log_extract/serializers.py:194
msgid "最新行数请输入整数"
msgstr ""

#: apps/log_extract/serializers.py:196
msgid "请保证最新行数大于0"
msgstr ""

#: apps/log_extract/serializers.py:201 apps/log_extract/serializers.py:205
msgid "过滤条件包含空关键词"
msgstr ""

#: apps/log_extract/serializers.py:211 apps/log_extract/serializers.py:215
#: apps/log_extract/serializers.py:233 apps/log_extract/serializers.py:240
#: apps/log_extract/serializers.py:338
msgid "下载任务ID"
msgstr ""

#: apps/log_extract/serializers.py:216
msgid "是否直接返回url"
msgstr ""

#: apps/log_extract/serializers.py:220 apps/log_search/serializers.py:384
msgid "下载任务ID列表"
msgstr ""

#: apps/log_extract/serializers.py:228
msgid "任务列表类型错误,请输入正确的整型数组"
msgstr ""

#: apps/log_extract/serializers.py:234
msgid "当前下载状态"
msgstr ""

#: apps/log_extract/serializers.py:245 apps/log_search/serializers.py:319
msgid "实例名称"
msgstr ""

#: apps/log_extract/serializers.py:246
msgid "对象ID"
msgstr ""

#: apps/log_extract/serializers.py:279 apps/log_search/models.py:665
msgid "用户组ID"
msgstr ""

#: apps/log_extract/serializers.py:280
msgid "授权的业务ID"
msgstr ""

#: apps/log_extract/serializers.py:282
msgid "授权的模板实例列表"
msgstr ""

#: apps/log_extract/serializers.py:283
msgid "授权的目录列表"
msgstr ""

#: apps/log_extract/serializers.py:284
msgid "授权的文件类型"
msgstr ""

#: apps/log_extract/serializers.py:303
msgid "请指定正确的目标选择类型"
msgstr ""

#: apps/log_extract/serializers.py:309
msgid "目录请以 '/' 结尾"
msgstr ""

#: apps/log_extract/serializers.py:314
msgid "文件后缀格式有误"
msgstr ""

#: apps/log_extract/serializers.py:317
msgid "文件后缀不需要使用.开头"
msgstr ""

#: apps/log_extract/serializers.py:341
msgid "任务状态"
msgstr ""

#: apps/log_extract/serializers.py:345
msgid "任务创建者"
msgstr ""

#: apps/log_extract/serializers.py:400
msgid "加密的目标文件名"
msgstr ""

#: apps/log_extract/serializers.py:410 apps/log_search/models.py:828
#: apps/models.py:205
msgid "创建者"
msgstr ""

#: apps/log_extract/serializers.py:434
msgid "中转机列表"
msgstr ""

#: apps/log_extract/tasks/extract.py:383
msgid "文件提取开始: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:384
msgid "文件打包ka: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:389
msgid "确认文件打包结果: {}, 确认次数: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:392
msgid "文件打包成功: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:395
msgid "文件打包异常(超出设定次数: {})"
msgstr ""

#: apps/log_extract/tasks/extract.py:397
msgid "文件分发中: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:401
msgid "确认文件分发结果: {}, 确认次数: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:404
msgid "文件分发成功: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:407
msgid "文件分发异常(超出设定次数: {})"
msgstr ""

#: apps/log_extract/tasks/extract.py:409
msgid "文件上传至cos中: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:413
msgid "确认文件上传至cos结果: {}, 确认次数: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:416
msgid "文件上传至cos成功: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:419
msgid "文件上传至cos异常(超出设定次数: {})"
msgstr ""

#: apps/log_extract/tasks/extract.py:424
msgid "文件上传至bkrepo准备中: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:426
msgid "文件上传至bkrepo成功: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:428
msgid "文件提取失败: {}, 失败原因: {}"
msgstr ""

#: apps/log_extract/tasks/extract.py:433
msgid "文件提取成功: {}"
msgstr ""

#: apps/log_measure/exceptions.py:32
msgid "容量管理模块异常"
msgstr ""

#: apps/log_measure/exceptions.py:42
msgid "集群权限异常"
msgstr ""

#: apps/log_measure/exceptions.py:47
msgid "ES连接失败"
msgstr ""

#: apps/log_measure/handlers/metric_collectors/business.py:32
#: apps/log_measure/handlers/metrics.py:271
msgid "活跃业务"
msgstr ""

#: apps/log_measure/handlers/metric_collectors/cluster.py:31
#: apps/log_measure/handlers/metrics.py:425
msgid "集群健康度"
msgstr ""

#: apps/log_measure/handlers/metric_collectors/collect.py:36
#: apps/log_measure/handlers/metrics.py:313
msgid "采集配置"
msgstr ""

#: apps/log_measure/handlers/metric_collectors/es.py:1022
msgid "es 监控信息"
msgstr ""

#: apps/log_measure/handlers/metric_collectors/grafana.py:33
#: apps/log_measure/handlers/metrics.py:538
msgid "Grafana 仪表盘"
msgstr ""

#: apps/log_measure/handlers/metric_collectors/third_party.py:32
#: apps/log_measure/handlers/metrics.py:400 apps/log_search/models.py:149
#: config/default.py:522 config/default.py:619
msgid "第三方ES"
msgstr ""

#: apps/log_measure/handlers/metric_collectors/user.py:38
#: apps/log_measure/handlers/metrics.py:300
msgid "活跃用户"
msgstr ""

#: apps/log_measure/handlers/metric_collectors/user.py:64
msgid "用户检索历史"
msgstr ""

#: apps/log_measure/handlers/metrics.py:338
msgid "采集主机"
msgstr ""

#: apps/log_measure/models.py:28 apps/log_search/models.py:162
#: apps/log_search/models.py:230 apps/log_search/models.py:290
#: apps/log_search/models.py:663 apps/log_search/models.py:697
#: apps/log_search/serializers.py:197 apps/log_search/serializers.py:337
#: apps/log_search/serializers.py:358 apps/log_trace/serializers.py:41
msgid "项目ID"
msgstr ""

#: apps/log_measure/models.py:29 apps/log_search/models.py:288
#: apps/log_search/models.py:574 apps/log_search/models.py:593
#: apps/log_search/models.py:604 apps/log_search/serializers.py:338
msgid "索引集ID"
msgstr ""

#: apps/log_measure/models.py:30
msgid "访问次数"
msgstr ""

#: apps/log_measure/models.py:31
msgid "访问日期"
msgstr ""

#: apps/log_measure/models.py:34 apps/log_measure/models.py:35
msgid "索引集搜索统计"
msgstr ""

#: apps/log_search/constants.py:40 config/default.py:463
msgid "trace"
msgstr ""

#: apps/log_search/constants.py:41
msgid "回溯中"
msgstr ""

#: apps/log_search/constants.py:42
msgid "回溯日志"
msgstr ""

#: apps/log_search/constants.py:43 apps/log_search/constants.py:261
msgid "无数据"
msgstr ""

#: apps/log_search/constants.py:44
msgid "有延迟"
msgstr ""

#: apps/log_search/constants.py:45 config/default.py:515 config/default.py:614
msgid "计算平台"
msgstr ""

#: apps/log_search/constants.py:46
msgid "BCS"
msgstr ""

#: apps/log_search/constants.py:59
msgid "red"
msgstr ""

#: apps/log_search/constants.py:60
msgid "yellow"
msgstr ""

#: apps/log_search/constants.py:61
msgid "blue"
msgstr ""

#: apps/log_search/constants.py:62
msgid "green"
msgstr ""

#: apps/log_search/constants.py:63
msgid "gray"
msgstr ""

#: apps/log_search/constants.py:186
msgid "必须"
msgstr ""

#: apps/log_search/constants.py:186
msgid "建议"
msgstr ""

#: apps/log_search/constants.py:186
msgid "用户自定义"
msgstr ""

#: apps/log_search/constants.py:197
msgid "字段缺失"
msgstr ""

#: apps/log_search/constants.py:198
msgid "数据类型不匹配"
msgstr ""

#: apps/log_search/constants.py:221
msgid "业务"
msgstr ""

#: apps/log_search/constants.py:232
msgid "服务模版"
msgstr ""

#: apps/log_search/constants.py:233
msgid "集群模版"
msgstr ""

#: apps/log_search/constants.py:257
msgid "异常（未知）"
msgstr ""

#: apps/log_search/constants.py:259
msgid "关闭"
msgstr ""

#: apps/log_search/constants.py:260
msgid "Agent未安装"
msgstr ""

#: apps/log_search/constants.py:283 apps/log_search/constants.py:555
#: apps/log_trace/constants.py:90 apps/log_trace/handlers/proto/log.py:62
#: apps/log_trace/handlers/proto/log.py:115
#: apps/log_trace/handlers/proto/otlp.py:59
#: apps/log_trace/handlers/trace_field_handlers.py:47
#: apps/log_trace/handlers/trace_field_handlers.py:91
#: apps/log_trace/handlers/trace_field_handlers.py:138
#: apps/log_trace/handlers/trace_field_handlers.py:156
#: apps/log_trace/handlers/trace_field_handlers.py:187
msgid "服务"
msgstr ""

#: apps/log_search/constants.py:311
msgid "分隔符列表"
msgstr ""

#: apps/log_search/constants.py:312
msgid "编码格式"
msgstr ""

#: apps/log_search/constants.py:313
msgid "数据保留时间"
msgstr ""

#: apps/log_search/constants.py:314
msgid "字段提取方式"
msgstr ""

#: apps/log_search/constants.py:315
msgid "字段类型"
msgstr ""

#: apps/log_search/constants.py:317
msgid "内置字段"
msgstr ""

#: apps/log_search/constants.py:318 apps/log_search/models.py:315
msgid "时间字段类型"
msgstr ""

#: apps/log_search/constants.py:319 apps/log_search/models.py:316
msgid "时间字段单位"
msgstr ""

#: apps/log_search/constants.py:320
msgid "日志来源类型"
msgstr ""

#: apps/log_search/constants.py:321
msgid "日志聚类敏感度"
msgstr ""

#: apps/log_search/constants.py:322
msgid "日志聚类同比配置"
msgstr ""

#: apps/log_search/constants.py:323 config/default.py:527
msgid "自定义上报"
msgstr ""

#: apps/log_search/constants.py:346
msgid "容器日志上报"
msgstr ""

#: apps/log_search/constants.py:347
msgid "otlpTrace上报"
msgstr ""

#: apps/log_search/constants.py:348
msgid "otlp日志上报"
msgstr ""

#: apps/log_search/constants.py:356
msgid ""
"\n"
"# 日志自定义上报\n"
"日志自定义上报适用于自行上报的服务或者场景，如下\n"
"\n"
"- 容器日志采集\n"
"- 服务自定义上报\n"
"- 其他\n"
"        "
msgstr ""

#: apps/log_search/constants.py:366
#, python-brace-format
msgid ""
"\n"
"# 注意事项\n"
"\n"
"- sdk内注入的属性类型应该一致，不然会出现入库失败的情况\n"
"\n"
"# 使用方法\n"
"\n"
"\n"
"不同云区域的自定义上报服务地址\n"
"\n"
"{otlp_report_config}\n"
"\n"
"[opentelemetry官方文档](https://opentelemetry.io/)\n"
"\n"
"# SDK配置\n"
"\n"
"python\n"
"\n"
"[python-SDK](https://github.com/open-telemetry/opentelemetry-python)\n"
"\n"
"    from opentelemetry import trace\n"
"    from opentelemetry.sdk.trace import TracerProvider\n"
"    from opentelemetry.sdk.trace.export import BatchSpanProcessor\n"
"    from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import "
"OTLPSpanExporter\n"
"    otlp_exporter = OTLPSpanExporter(endpoint=\"grpc 上报服务地址)    \n"
"    span_processor = BatchSpanProcessor(otlp_exporter)\n"
"    tracer_provider = TracerProvider(\n"
"        resource=Resource.create(\n"
"            {{\n"
"                \"service.name\": \"你的服务名称\",\n"
"                \"bk_data_id\": {{{{bk_data_id}}}},\n"
"            }}\n"
"        )\n"
"    )\n"
"    tracer_provider.add_span_processor(span_processor)\n"
"    trace.set_tracer_provider(tracer_provider)\n"
"    \n"
"        "
msgstr ""

#: apps/log_search/constants.py:406
#, python-brace-format
msgid ""
"\n"
"# 注意事项\n"
"# 使用方法\n"
"不同云区域的自定义上报服务地址\n"
"\n"
"{otlp_report_config}\n"
"\n"
"\n"
"[opentelemetry官方文档](https://opentelemetry.io/)\n"
msgstr ""

#: apps/log_search/constants.py:427
msgid "行日志文件"
msgstr ""

#: apps/log_search/constants.py:428
msgid "段日志文件"
msgstr ""

#: apps/log_search/constants.py:429
msgid "win event日志"
msgstr ""

#: apps/log_search/constants.py:452
msgid "JSON"
msgstr ""

#: apps/log_search/constants.py:462
msgid "win event"
msgstr ""

#: apps/log_search/constants.py:551
msgid "操作系统"
msgstr ""

#: apps/log_search/constants.py:556
msgid "服务模块"
msgstr ""

#: apps/log_search/constants.py:562
msgid "业务应用"
msgstr ""

#: apps/log_search/constants.py:625
msgid "包含"
msgstr ""

#: apps/log_search/constants.py:626
msgid "不包含"
msgstr ""

#: apps/log_search/constants.py:639
msgid "字符串"
msgstr ""

#: apps/log_search/constants.py:656
msgid "竖线(|)"
msgstr ""

#: apps/log_search/constants.py:657
msgid "逗号(,)"
msgstr ""

#: apps/log_search/constants.py:658
msgid "反引号(`)"
msgstr ""

#: apps/log_search/constants.py:659
msgid "空格"
msgstr ""

#: apps/log_search/constants.py:660
msgid "分号(;)"
msgstr ""

#: apps/log_search/constants.py:687
msgid "1天"
msgstr ""

#: apps/log_search/constants.py:688
msgid "3天"
msgstr ""

#: apps/log_search/constants.py:689
msgid "7天"
msgstr ""

#: apps/log_search/constants.py:690
msgid "14天"
msgstr ""

#: apps/log_search/constants.py:691
msgid "30天"
msgstr ""

#: apps/log_search/constants.py:704
msgid "date"
msgstr ""

#: apps/log_search/constants.py:705
msgid "long"
msgstr ""

#: apps/log_search/constants.py:718
msgid "second"
msgstr ""

#: apps/log_search/constants.py:718
msgid "millisecond"
msgstr ""

#: apps/log_search/constants.py:718
msgid "microsecond"
msgstr ""

#: apps/log_search/exceptions.py:35
msgid "参数校验异常：项目ID不存在"
msgstr ""

#: apps/log_search/exceptions.py:43
msgid "管理模块异常"
msgstr ""

#: apps/log_search/exceptions.py:53
msgid "接入场景异常"
msgstr ""

#: apps/log_search/exceptions.py:58
#, python-brace-format
msgid "暂不支持\"{scenario_id}\"场景的操作"
msgstr ""

#: apps/log_search/exceptions.py:63
#, python-brace-format
msgid "探测ES连通性失败,失败原因：\"{es_fail_reason}\""
msgstr ""

#: apps/log_search/exceptions.py:73
#, python-brace-format
msgid "EsClient查询index失败,失败原因:\"{es_fail_reason}\""
msgstr ""

#: apps/log_search/exceptions.py:81
msgid "数据源模块异常"
msgstr ""

#: apps/log_search/exceptions.py:86
#, python-brace-format
msgid "该项目下已存在名称为\"{source_name}\"的数据源"
msgstr ""

#: apps/log_search/exceptions.py:91
msgid "数据源不存在"
msgstr ""

#: apps/log_search/exceptions.py:96
msgid "此数据源不允许编辑"
msgstr ""

#: apps/log_search/exceptions.py:101
msgid "ES服务器连接异常"
msgstr ""

#: apps/log_search/exceptions.py:106
msgid "此数据源已有关联的索引集，无法直接删除"
msgstr ""

#: apps/log_search/exceptions.py:116
msgid "索引集模块异常"
msgstr ""

#: apps/log_search/exceptions.py:121
msgid "索引集不存在"
msgstr ""

#: apps/log_search/exceptions.py:126
#, python-brace-format
msgid "索引\"{result_table_id}\"在数据平台中不存在"
msgstr ""

#: apps/log_search/exceptions.py:131
#, python-brace-format
msgid "该项目下已存在名称为\"{index_set_name}\"的数据集"
msgstr ""

#: apps/log_search/exceptions.py:136
#, python-brace-format
msgid "索引\"{result_table_id}\"在此ES中不存在"
msgstr ""

#: apps/log_search/exceptions.py:141
#, python-brace-format
msgid "该索引集下已存在\"{result_table_id}\"结果表"
msgstr ""

#: apps/log_search/exceptions.py:146
msgid "索引字段不一致，无法添加索引到同一个索引集中"
msgstr ""

#: apps/log_search/exceptions.py:151
msgid "索引已存在，请勿添加相同索引"
msgstr ""

#: apps/log_search/exceptions.py:156
msgid "索引集不支持跨业务"
msgstr ""

#: apps/log_search/exceptions.py:161
msgid "获取结果表信息失败"
msgstr ""

#: apps/log_search/exceptions.py:166
msgid "索引集不支持跨集群"
msgstr ""

#: apps/log_search/exceptions.py:171
msgid "索引列表数据异常"
msgstr ""

#: apps/log_search/exceptions.py:176
msgid "索引列表不满足Trace要求"
msgstr ""

#: apps/log_search/exceptions.py:181
msgid "项目ID不存在无法搜索TRACE类型索引集"
msgstr ""

#: apps/log_search/exceptions.py:186
#, python-brace-format
msgid "索引字段{field_type}类型不一致，无法添加索引到同一个索引集中"
msgstr ""

#: apps/log_search/exceptions.py:191
msgid "索引缺少时间字段，无法添加索引到同一个索引集中"
msgstr ""

#: apps/log_search/exceptions.py:196
msgid "索引时间字段不同，无法添加索引到同一个索引集中"
msgstr ""

#: apps/log_search/exceptions.py:201
msgid "索引时间字段类型不同，无法添加索引到同一个索引集中"
msgstr ""

#: apps/log_search/exceptions.py:206
#, python-brace-format
msgid "索引{result_table_id} mapping信息查询结果为空"
msgstr ""

#: apps/log_search/exceptions.py:211
msgid "您没有此索引集的编辑权限"
msgstr ""

#: apps/log_search/exceptions.py:216
msgid "索引集不能为空集"
msgstr ""

#: apps/log_search/exceptions.py:221
msgid "索引集中的索引字段不能有冲突"
msgstr ""

#: apps/log_search/exceptions.py:231
msgid "检索模块异常"
msgstr ""

#: apps/log_search/exceptions.py:241
#, python-brace-format
msgid "找不到索引集\"{index_set_id}\"的时间字段信息，无法进行检索"
msgstr ""

#: apps/log_search/exceptions.py:246
msgid "将时间字段转换成handler接口可用格式失败"
msgstr ""

#: apps/log_search/exceptions.py:251
msgid "参数中没有keyword"
msgstr ""

#: apps/log_search/exceptions.py:256
msgid "参数中没有开始指针"
msgstr ""

#: apps/log_search/exceptions.py:261
msgid "场景选择错误"
msgstr ""

#: apps/log_search/exceptions.py:266
msgid "查询上下文需要gseindex相关信息"
msgstr ""

#: apps/log_search/exceptions.py:271
msgid "链接到ES数据源失败"
msgstr ""

#: apps/log_search/exceptions.py:276
msgid "添加用户index字段时缺少相关参数"
msgstr ""

#: apps/log_search/exceptions.py:281
#, python-brace-format
msgid "查询ES中索引下的fields字段失败,失败原因{fail_reason}"
msgstr ""

#: apps/log_search/exceptions.py:296
msgid "不需要输出"
msgstr ""

#: apps/log_search/exceptions.py:301
#, python-brace-format
msgid "search dict empty错误{search_dict}"
msgstr ""

#: apps/log_search/exceptions.py:306
#, python-brace-format
msgid "该字段{sort_item}不支持排序"
msgstr ""

#: apps/log_search/exceptions.py:311
#, python-brace-format
msgid "拉取schema错误{index}"
msgstr ""

#: apps/log_search/exceptions.py:316
msgid "Trace查询time field不存在"
msgstr ""

#: apps/log_search/exceptions.py:321
#, python-brace-format
msgid "超出最大查询数量：{size}"
msgstr ""

#: apps/log_search/exceptions.py:326
msgid "未知的时间字段类型"
msgstr ""

#: apps/log_search/exceptions.py:331
msgid "未知的时间字段/时间字段类型/时间字段单位"
msgstr ""

#: apps/log_search/exceptions.py:336
#, python-brace-format
msgid "用户无结果表 {result_tables} 的管理权限，无法添加到索引集中"
msgstr ""

#: apps/log_search/exceptions.py:341
msgid "此索引没有找到时间字段类型"
msgstr ""

#: apps/log_search/exceptions.py:346
msgid "收藏查询不存在"
msgstr ""

#: apps/log_search/exceptions.py:351
#, python-brace-format
msgid "索引集【{index_set_id}】聚合查询异常：{err}"
msgstr ""

#: apps/log_search/exceptions.py:361
msgid "对应索引集缺少异步导出必备fields{}"
msgstr ""

#: apps/log_search/exceptions.py:366
#, python-brace-format
msgid "超过异步导出最大条数限制{max_async_export_count}"
msgstr ""

#: apps/log_search/exceptions.py:371
#, python-brace-format
msgid "无法找到{template_name}{language}相关模板"
msgstr ""

#: apps/log_search/exceptions.py:381
msgid "请升级blueapps至最新版本"
msgstr ""

#: apps/log_search/exceptions.py:386
msgid "获取JWT信息异常"
msgstr ""

#: apps/log_search/exceptions.py:391
msgid "JWT校验失败"
msgstr ""

#: apps/log_search/exceptions.py:396
msgid "配置中menu对象异常"
msgstr ""

#: apps/log_search/handlers/biz.py:520
msgid "无法识别节点"
msgstr ""

#: apps/log_search/handlers/index_set.py:175 apps/log_search/models.py:567
msgid "审批中"
msgstr ""

#: apps/log_search/handlers/search/mapping_handlers.py:632
msgid "必须path或者logfile字段"
msgstr ""

#: apps/log_search/handlers/search/mapping_handlers.py:637
msgid "必须_iteration_idx字段"
msgstr ""

#: apps/log_search/handlers/search/mapping_handlers.py:643
msgid "必须ip或者container_id字段"
msgstr ""

#: apps/log_search/handlers/search/mapping_handlers.py:648
msgid "必须serverIp字段"
msgstr ""

#: apps/log_search/handlers/search/mapping_handlers.py:651
msgid "必须path字段"
msgstr ""

#: apps/log_search/handlers/search/mapping_handlers.py:654
msgid "必须iterationIndex或者_iteration_idx字段"
msgstr ""

#: apps/log_search/handlers/search/mapping_handlers.py:656
msgid "必须gseindex或者gseIndex字段"
msgstr ""

#: apps/log_search/handlers/search/mapping_handlers.py:721
msgid "【异步导出功能尚未开放】"
msgstr ""

#: apps/log_search/handlers/search/mapping_handlers.py:748
#, python-brace-format
msgid "缺少必备字段: {async_fields} or {async_container_fields}"
msgstr ""

#: apps/log_search/handlers/search/mapping_handlers.py:752
#, python-brace-format
msgid "缺少必备字段: {async_fields}"
msgstr ""

#: apps/log_search/migrations/0002_bklog_initial.py:38
msgid "ES"
msgstr ""

#: apps/log_search/migrations/0002_bklog_initial.py:39
msgid "日志文件"
msgstr ""

#: apps/log_search/migrations/0002_bklog_initial.py:59
msgid "阈值监控"
msgstr ""

#: apps/log_search/models.py:80
msgid "配置标识"
msgstr ""

#: apps/log_search/models.py:81
msgid "具体配置"
msgstr ""

#: apps/log_search/models.py:128 apps/log_search/models.py:129
msgid "00_全局配置"
msgstr ""

#: apps/log_search/models.py:147 config/default.py:607
msgid "采集接入"
msgstr ""

#: apps/log_search/models.py:163
msgid "项目名称"
msgstr ""

#: apps/log_search/models.py:168
msgid "是否可以使用ip快选"
msgstr ""

#: apps/log_search/models.py:169
msgid "是否为 CMDB V3 业务"
msgstr ""

#: apps/log_search/models.py:170
msgid "是否混合使用CMDB V3 业务"
msgstr ""

#: apps/log_search/models.py:171
msgid "功能白名单"
msgstr ""

#: apps/log_search/models.py:218
msgid "项目列表"
msgstr ""

#: apps/log_search/models.py:219
msgid "02_项目列表"
msgstr ""

#: apps/log_search/models.py:227 apps/log_search/models.py:296
#: apps/log_search/serializers.py:77 apps/log_search/serializers.py:93
msgid "数据源ID"
msgstr ""

#: apps/log_search/models.py:229 apps/log_search/models.py:294
msgid "接入场景标识"
msgstr ""

#: apps/log_search/models.py:231
msgid "属性"
msgstr ""

#: apps/log_search/models.py:232 apps/log_search/models.py:297
msgid "顺序"
msgstr ""

#: apps/log_search/models.py:233
msgid "是否可以编辑"
msgstr ""

#: apps/log_search/models.py:252
#, python-brace-format
msgid "此项目[{self.project_id}]下已存在"
msgstr ""

#: apps/log_search/models.py:271
msgid "暂不支持除ES自定义场景意外的其它场景"
msgstr ""

#: apps/log_search/models.py:275
msgid "ES集群配置"
msgstr ""

#: apps/log_search/models.py:276
msgid "03_ES集群配置"
msgstr ""

#: apps/log_search/models.py:292
msgid "绑定的数据平台项目ID"
msgstr ""

#: apps/log_search/models.py:293
msgid "绑定Transfer采集ID"
msgstr ""

#: apps/log_search/models.py:295 apps/log_search/serializers.py:114
msgid "存储集群ID"
msgstr ""

#: apps/log_search/models.py:302
msgid "是否通过"
msgstr ""

#: apps/log_search/models.py:303
msgid "预查询描述"
msgstr ""

#: apps/log_search/models.py:306
msgid "字段快照"
msgstr ""

#: apps/log_search/models.py:309
msgid "是否trace"
msgstr ""

#: apps/log_search/models.py:311
msgid "来源系统"
msgstr ""

#: apps/log_search/models.py:314 apps/log_search/models.py:578
msgid "时间字段"
msgstr ""

#: apps/log_search/models.py:317
msgid "标签id记录"
msgstr ""

#: apps/log_search/models.py:551
msgid "索引集配置"
msgstr ""

#: apps/log_search/models.py:552
msgid "21_索引集配置"
msgstr ""

#: apps/log_search/models.py:569
msgid "拒绝"
msgstr ""

#: apps/log_search/models.py:570
msgid "异常"
msgstr ""

#: apps/log_search/models.py:573
msgid "索引ID"
msgstr ""

#: apps/log_search/models.py:576
msgid "结果表"
msgstr ""

#: apps/log_search/models.py:577
msgid "结果表名称"
msgstr ""

#: apps/log_search/models.py:579
msgid "审核状态"
msgstr ""

#: apps/log_search/models.py:588
msgid "索引集数据"
msgstr ""

#: apps/log_search/models.py:589
msgid "22_索引集详情"
msgstr ""

#: apps/log_search/models.py:595 apps/log_search/serializers.py:211
msgid "排序规则"
msgstr ""

#: apps/log_search/models.py:596
msgid "应用范围"
msgstr ""

#: apps/log_search/models.py:599
msgid "索引集自定义显示"
msgstr ""

#: apps/log_search/models.py:600
msgid "31_搜索-索引集自定义显示"
msgstr ""

#: apps/log_search/models.py:605
msgid "检索条件"
msgstr ""

#: apps/log_search/models.py:606
msgid "检索类型"
msgstr ""

#: apps/log_search/models.py:607
msgid "查询耗时"
msgstr ""

#: apps/log_search/models.py:608
msgid "排序"
msgstr ""

#: apps/log_search/models.py:611
msgid "索引集用户检索记录"
msgstr ""

#: apps/log_search/models.py:612
msgid "32_搜索-索引集用户检索记录"
msgstr ""

#: apps/log_search/models.py:635
msgid "用户组"
msgstr ""

#: apps/log_search/models.py:636
msgid "资源"
msgstr ""

#: apps/log_search/models.py:649
msgid "待处理"
msgstr ""

#: apps/log_search/models.py:650
msgid "同步完成"
msgstr ""

#: apps/log_search/models.py:651
msgid "同步失败"
msgstr ""

#: apps/log_search/models.py:664
msgid "变更类型"
msgstr ""

#: apps/log_search/models.py:666
msgid "资源类型"
msgstr ""

#: apps/log_search/models.py:667
msgid "范围"
msgstr ""

#: apps/log_search/models.py:668
msgid "同步状态"
msgstr ""

#: apps/log_search/models.py:669
msgid "同步时间"
msgstr ""

#: apps/log_search/models.py:689
msgid "资源变更任务"
msgstr ""

#: apps/log_search/models.py:690
msgid "23_资源变更任务"
msgstr ""

#: apps/log_search/models.py:696
msgid "用户检索历史记录ID"
msgstr ""

#: apps/log_search/models.py:698 apps/log_search/serializers.py:339
msgid "收藏描述"
msgstr ""

#: apps/log_search/models.py:701 apps/log_search/models.py:711
msgid "检索收藏记录"
msgstr ""

#: apps/log_search/models.py:702 apps/log_search/models.py:712
msgid "33_搜索-检索收藏记录"
msgstr ""

#: apps/log_search/models.py:706
msgid "id"
msgstr ""

#: apps/log_search/models.py:708
msgid "用户name"
msgstr ""

#: apps/log_search/models.py:736
msgid "标签id"
msgstr ""

#: apps/log_search/models.py:737
msgid "标签名称"
msgstr ""

#: apps/log_search/models.py:738
msgid "配色"
msgstr ""

#: apps/log_search/models.py:741 apps/log_search/models.py:742
msgid "标签表"
msgstr ""

#: apps/log_search/models.py:764
msgid "检索请求参数"
msgstr ""

#: apps/log_search/models.py:765
msgid "异步导出排序字段"
msgstr ""

#: apps/log_search/models.py:766 apps/log_search/serializers.py:56
#: apps/log_search/serializers.py:76 apps/log_search/serializers.py:92
#: apps/log_search/serializers.py:113
msgid "接入场景"
msgstr ""

#: apps/log_search/models.py:768
msgid "异步导出结果"
msgstr ""

#: apps/log_search/models.py:769
msgid "异步导出异常原因"
msgstr ""

#: apps/log_search/models.py:770
msgid "文件名"
msgstr ""

#: apps/log_search/models.py:771
msgid "文件大小"
msgstr ""

#: apps/log_search/models.py:772
msgid "下载地址"
msgstr ""

#: apps/log_search/models.py:773
msgid "是否被清理"
msgstr ""

#: apps/log_search/models.py:774
msgid "导出状态"
msgstr ""

#: apps/log_search/models.py:775
msgid "导出选择请求时间"
msgstr ""

#: apps/log_search/models.py:776
msgid "导出选择结束时间"
msgstr ""

#: apps/log_search/models.py:777
msgid "导出类型"
msgstr ""

#: apps/log_search/models.py:779
msgid "任务完成时间"
msgstr ""

#: apps/log_search/models.py:783
msgid "导出任务"
msgstr ""

#: apps/log_search/models.py:784
msgid "42_导出任务"
msgstr ""

#: apps/log_search/models.py:793
msgid "模板路径"
msgstr ""

#: apps/log_search/models.py:794
msgid "语言级别"
msgstr ""

#: apps/log_search/models.py:823
msgid "邮件模板"
msgstr ""

#: apps/log_search/models.py:824
msgid "43_邮件模板"
msgstr ""

#: apps/log_search/models.py:829
msgid "用户meta配置"
msgstr ""

#: apps/log_search/models.py:833
msgid "用户元配置"
msgstr ""

#: apps/log_search/models.py:834
msgid "44_用户元配置"
msgstr ""

#: apps/log_search/serializers.py:46
msgid "分页大小"
msgstr ""

#: apps/log_search/serializers.py:59
msgid "索引"
msgstr ""

#: apps/log_search/serializers.py:66
msgid "业务ID不能为空"
msgstr ""

#: apps/log_search/serializers.py:69 apps/log_search/serializers.py:85
#: apps/log_search/serializers.py:100 apps/log_search/serializers.py:124
msgid "数据源ID不能为空"
msgstr ""

#: apps/log_search/serializers.py:75
msgid "索引列表"
msgstr ""

#: apps/log_search/serializers.py:87
msgid "indices不能为空"
msgstr ""

#: apps/log_search/serializers.py:115 apps/log_search/serializers.py:116
msgid "源索引"
msgstr ""

#: apps/log_search/serializers.py:117
msgid "待追加的索引"
msgstr ""

#: apps/log_search/serializers.py:131
msgid "源索引时间字段不能为空"
msgstr ""

#: apps/log_search/serializers.py:133
msgid "源索引时间字段类型不能为空"
msgstr ""

#: apps/log_search/serializers.py:137
msgid "待追加索引时间字段不能为空"
msgstr ""

#: apps/log_search/serializers.py:139
msgid "待追加索引时间字段类型不能为空"
msgstr ""

#: apps/log_search/serializers.py:180
msgid "最大只支持查询7天数据"
msgstr ""

#: apps/log_search/serializers.py:204 apps/log_search/serializers.py:365
#: apps/log_trace/serializers.py:45
msgid "请输入业务ID"
msgstr ""

#: apps/log_search/serializers.py:216
msgid "sort_list参数格式有误"
msgstr ""

#: apps/log_search/serializers.py:219
msgid "排序规则只支持升序asc或降序desc"
msgstr ""

#: apps/log_search/serializers.py:251
msgid "时间范围"
msgstr ""

#: apps/log_search/serializers.py:252
msgid "起始时间"
msgstr ""

#: apps/log_search/serializers.py:254
msgid "检索模块ip等信息"
msgstr ""

#: apps/log_search/serializers.py:255
msgid "搜索条件"
msgstr ""

#: apps/log_search/serializers.py:256
msgid "检索开始 offset"
msgstr ""

#: apps/log_search/serializers.py:257
msgid "检索结果大小"
msgstr ""

#: apps/log_search/serializers.py:258
msgid "匹配规则"
msgstr ""

#: apps/log_search/serializers.py:276
msgid "是否展示业务全量导出历史"
msgstr ""

#: apps/log_search/serializers.py:281
msgid "ES HOST"
msgstr ""

#: apps/log_search/serializers.py:282
msgid "ES 端口"
msgstr ""

#: apps/log_search/serializers.py:283
msgid "ES 用户"
msgstr ""

#: apps/log_search/serializers.py:284
msgid "ES 密码"
msgstr ""

#: apps/log_search/serializers.py:292
msgid "主机IP"
msgstr ""

#: apps/log_search/serializers.py:293
msgid "云区域ID"
msgstr ""

#: apps/log_search/serializers.py:318
msgid "实例id"
msgstr ""

#: apps/log_search/serializers.py:320
msgid "类型id"
msgstr ""

#: apps/log_search/serializers.py:346
msgid "索引集ID不能为空"
msgstr ""

#: apps/log_search/serializers.py:348
msgid "项目id不能为空"
msgstr ""

#: apps/log_search/serializers.py:375
msgid "集群id"
msgstr ""

#: apps/log_search/serializers.py:376
msgid "容器id"
msgstr ""

#: apps/log_search/serializers.py:380 apps/log_search/serializers.py:385
msgid "模版类型"
msgstr ""

#: apps/log_search/serializers.py:393
msgid "类型错误,请输入正确的整型数组"
msgstr ""

#: apps/log_search/serializers.py:398
msgid "动态分组ID列表"
msgstr ""

#: apps/log_search/serializers.py:398
msgid "动态分组ID"
msgstr ""

#: apps/log_search/tasks/async_export.py:294
#, python-brace-format
msgid "【{platform}】{index_set_name} 检索导出"
msgstr ""

#: apps/log_search/tasks/async_export.py:295
#, python-brace-format
msgid "【{platform}】{index_set_name} 检索导出失败"
msgstr ""

#: apps/log_search/views/index_set_views.py:111
msgid "集群ID不能为空"
msgstr ""

#: apps/log_search/views/index_set_views.py:119
#, python-brace-format
msgid "业务({bk_biz_id})对应的项目信息不存在"
msgstr ""

#: apps/log_search/views/index_set_views.py:122
msgid "参数 bk_biz_id 和 project_id 请至少提供一项"
msgstr ""

#: apps/log_search/views/meta_views.py:395
#: apps/log_search/views/meta_views.py:420
msgid "username 不能为空"
msgstr ""

#: apps/log_search/views/meta_views.py:470
msgid "请选择语言"
msgstr ""

#: apps/log_search/views/meta_views.py:519
msgid "project_id不能为空"
msgstr ""

#: apps/log_search/views/search_views.py:514
#, python-brace-format
msgid ""
"任务提交成功，预估等待时间{time}分钟,系统处理后将通过{notify_type_name}通知，"
"请留意！"
msgstr ""

#: apps/log_search/views/search_views.py:682
#: apps/log_search/views/search_views.py:745
msgid "scope取值范围：default、search_context"
msgstr ""

#: apps/log_search/views/search_views.py:781
msgid "is"
msgstr ""

#: apps/log_search/views/search_views.py:782
msgid "is one of"
msgstr ""

#: apps/log_search/views/search_views.py:783
msgid "is not"
msgstr ""

#: apps/log_search/views/search_views.py:784
msgid "is not one of"
msgstr ""

#: apps/log_search/views/search_views.py:785
msgid "大于"
msgstr ""

#: apps/log_search/views/search_views.py:786
msgid "大于等于"
msgstr ""

#: apps/log_search/views/search_views.py:787
msgid "小于"
msgstr ""

#: apps/log_search/views/search_views.py:788
msgid "小于等于"
msgstr ""

#: apps/log_search/views/search_views.py:789
msgid "exists"
msgstr ""

#: apps/log_search/views/search_views.py:789
msgid "确认字段已存在"
msgstr ""

#: apps/log_search/views/search_views.py:790
msgid "does not exists"
msgstr ""

#: apps/log_search/views/search_views.py:790
msgid "确认字段不存在"
msgstr ""

#: apps/log_trace/constants.py:29
msgid "日志平台协议"
msgstr ""

#: apps/log_trace/constants.py:30
msgid "opentelemetry协议"
msgstr ""

#: apps/log_trace/constants.py:42
msgid "关联"
msgstr ""

#: apps/log_trace/constants.py:42
msgid "取消关联"
msgstr ""

#: apps/log_trace/constants.py:50
msgid "traceID"
msgstr ""

#: apps/log_trace/constants.py:57
msgid "spanID"
msgstr ""

#: apps/log_trace/constants.py:64
msgid "SPAN"
msgstr ""

#: apps/log_trace/constants.py:89 apps/log_trace/handlers/proto/log.py:55
#: apps/log_trace/handlers/proto/log.py:107
#: apps/log_trace/handlers/trace_field_handlers.py:46
#: apps/log_trace/handlers/trace_field_handlers.py:89
#: apps/log_trace/handlers/trace_field_handlers.py:130
msgid "场景"
msgstr ""

#: apps/log_trace/constants.py:91 apps/log_trace/handlers/proto/log.py:85
#: apps/log_trace/handlers/proto/log.py:99
#: apps/log_trace/handlers/trace_field_handlers.py:48
#: apps/log_trace/handlers/trace_field_handlers.py:113
#: apps/log_trace/handlers/trace_field_handlers.py:122
msgid "返回码"
msgstr ""

#: apps/log_trace/constants.py:100
#: apps/log_trace/handlers/trace_field_handlers.py:50
msgid "日志"
msgstr ""

#: apps/log_trace/constants.py:114
msgid "数据时间"
msgstr ""

#: apps/log_trace/constants.py:133
msgid "平均聚合"
msgstr ""

#: apps/log_trace/constants.py:134
msgid "最大聚合"
msgstr ""

#: apps/log_trace/constants.py:135
msgid "最小聚合"
msgstr ""

#: apps/log_trace/constants.py:136
msgid "总和"
msgstr ""

#: apps/log_trace/constants.py:137
msgid "基数聚合"
msgstr ""

#: apps/log_trace/constants.py:138
msgid "统计汇总"
msgstr ""

#: apps/log_trace/exceptions.py:35
msgid "trace_id不存在"
msgstr ""

#: apps/log_trace/exceptions.py:40
msgid "trace日志没有找到根节点"
msgstr ""

#: apps/log_trace/exceptions.py:45
msgid "协议未支持"
msgstr ""

#: apps/log_trace/handlers/proto/log.py:33
msgid "成功"
msgstr ""

#: apps/log_trace/handlers/proto/log.py:59
#: apps/log_trace/handlers/proto/log.py:66
#: apps/log_trace/handlers/proto/log.py:73
#: apps/log_trace/handlers/proto/otlp.py:63
#: apps/log_trace/handlers/proto/otlp.py:70
#: apps/log_trace/handlers/proto/otlp.py:77
#: apps/log_trace/handlers/proto/otlp.py:84
#: apps/log_trace/handlers/trace_field_handlers.py:89
#: apps/log_trace/handlers/trace_field_handlers.py:95
#: apps/log_trace/handlers/trace_field_handlers.py:102
#: apps/log_trace/handlers/trace_field_handlers.py:160
#: apps/log_trace/handlers/trace_field_handlers.py:167
msgid "请选择"
msgstr ""

#: apps/log_trace/handlers/proto/log.py:69
#: apps/log_trace/handlers/trace_field_handlers.py:98
#: apps/log_trace/handlers/trace_field_handlers.py:163
msgid "操作名"
msgstr ""

#: apps/log_trace/handlers/proto/log.py:78
#: apps/log_trace/handlers/trace_field_handlers.py:107
#: apps/log_trace/handlers/trace_field_handlers.py:172
msgid "耗时min_max"
msgstr ""

#: apps/log_trace/handlers/proto/log.py:95
#: apps/log_trace/handlers/trace_field_handlers.py:118
#: apps/log_trace/handlers/trace_field_handlers.py:183
msgid "曲线图"
msgstr ""

#: apps/log_trace/handlers/proto/log.py:103
#: apps/log_trace/handlers/proto/log.py:111
#: apps/log_trace/handlers/trace_field_handlers.py:126
#: apps/log_trace/handlers/trace_field_handlers.py:134
msgid "耗时图"
msgstr ""

#: apps/log_trace/handlers/proto/log.py:119
#: apps/log_trace/handlers/proto/otlp.py:90
#: apps/log_trace/handlers/trace_field_handlers.py:142
#: apps/log_trace/handlers/trace_field_handlers.py:191
msgid "调用关系图"
msgstr ""

#: apps/log_trace/handlers/proto/log.py:125
#: apps/log_trace/handlers/proto/otlp.py:96
#: apps/log_trace/handlers/trace_field_handlers.py:148
#: apps/log_trace/handlers/trace_field_handlers.py:197
msgid "operationName调用关系"
msgstr ""

#: apps/log_trace/handlers/proto/otlp.py:66
msgid "名称"
msgstr ""

#: apps/log_trace/handlers/proto/otlp.py:80
msgid "trace状态"
msgstr ""

#: apps/log_trace/serializers.py:134
msgid "metric_field字段不能为空"
msgstr ""

#: apps/log_trace/views/trace_views.py:539
#: apps/log_trace/views/trace_views.py:568
msgid "scope取值范围：trace、trace_detail、trace_detail_log"
msgstr ""

#: apps/middlewares.py:131
msgid "S-mart应用仅支持正式环境部署！"
msgstr ""

#: apps/models.py:206
msgid "更新时间"
msgstr ""

#: apps/models.py:207
msgid "修改者"
msgstr ""

#: apps/models.py:266
msgid "删除时间"
msgstr ""

#: apps/models.py:267
msgid "删除者"
msgstr ""

#: apps/utils/cos.py:36
msgid "请设置腾讯云cos SecretId"
msgstr ""

#: apps/utils/cos.py:37
msgid "请设置腾讯云cos SecretKey"
msgstr ""

#: apps/utils/cos.py:38
msgid "请设置腾讯云cos 区域"
msgstr ""

#: apps/utils/cos.py:39
msgid "请设置腾讯云cos 桶名称"
msgstr ""

#: apps/utils/custom_report.py:26
#, python-brace-format
msgid "    云区域ID {cloud_id} {service}"
msgstr ""

#: apps/utils/drf.py:42
msgid "参数校验失败:\n"
msgstr ""

#: apps/utils/drf.py:76
#, python-brace-format
msgid ""
"\n"
"{prefix}第{index}项:"
msgstr ""

#: apps/utils/drf.py:109
msgid "参数校验失败，详情请查看返回的errors"
msgstr ""

#: apps/utils/pipline.py:58
#, python-brace-format
msgid "开始{name} pipeline_id=>{pipeline_id} node_id=>{node_id} "
msgstr ""

#: apps/utils/pipline.py:65 apps/utils/pipline.py:97
#, python-brace-format
msgid "{name}失败"
msgstr ""

#: apps/utils/pipline.py:68 apps/utils/pipline.py:100
#, python-brace-format
msgid "[{name}] {reason}"
msgstr ""

#: apps/utils/pipline.py:70 apps/utils/pipline.py:101
#, python-brace-format
msgid "{name}失败: {reason}"
msgstr ""

#: apps/utils/pipline.py:89
#, python-brace-format
msgid "开始轮询结果：{name} pipeline_id=>{pipeline_id} node_id=>{node_id} "
msgstr ""

#: apps/utils/pipline.py:128
msgid "定时时间"
msgstr ""

#: apps/utils/pipline.py:133
msgid "是否强制晚于当前时间"
msgstr ""

#: apps/utils/pipline.py:174
msgid "定时"
msgstr ""

#: bk_monitor/handler/monitor.py:94
msgid "数据源名称不能为空"
msgstr ""

#: bk_monitor/handler/monitor.py:125
#, python-brace-format
msgid ""
"data_id已创建却无法获取相关信息， 建议联系管理员处理 data_name -> {data_name}"
msgstr ""

#: bk_monitor/handler/monitor.py:127
msgid "data_id已创建却无法获取相关信息， 建议联系管理员处理"
msgstr ""

#: bk_monitor/handler/monitor.py:185
#, python-brace-format
msgid "f{key} data_name初始化异常，请检查"
msgstr ""

#: bk_monitor/handler/monitor.py:248
#, python-brace-format
msgid "{key} data对应数据源为空，请检查数据源"
msgstr ""

#: bk_monitor/handler/monitor.py:252
#, python-brace-format
msgid "{val} 数据源采集的数据为空"
msgstr ""

#: bk_monitor/models.py:12
msgid "时序数据"
msgstr ""

#: bk_monitor/models.py:13
msgid "事件数据"
msgstr ""

#: bk_monitor/models.py:15
msgid "监控数据源id"
msgstr ""

#: bk_monitor/models.py:17
msgid "指定业务id"
msgstr ""

#: bk_monitor/models.py:19
msgid "数据上报token"
msgstr ""

#: bk_monitor/models.py:20
msgid "数据源是否enable"
msgstr ""

#: bk_monitor/models.py:24 bk_monitor/models.py:25
msgid "运营数据上报配置"
msgstr ""

#: bk_monitor/utils/query.py:52
#, python-brace-format
msgid "{self.data_name} data_name 不存在， 请先初始化"
msgstr ""

#: config/default.py:451
msgid "检索"
msgstr ""

#: config/default.py:454 config/default.py:460
msgid "调用链"
msgstr ""

#: config/default.py:465
msgid "调用链列表"
msgstr ""

#: config/default.py:466
msgid "调用链详情"
msgstr ""

#: config/default.py:471
msgid "监控策略"
msgstr ""

#: config/default.py:474 config/default.py:480
msgid "仪表盘"
msgstr ""

#: config/default.py:483
msgid "仪表"
msgstr ""

#: config/default.py:485
msgid "默认仪表盘"
msgstr ""

#: config/default.py:486
msgid "新建仪表盘"
msgstr ""

#: config/default.py:487
msgid "新建目录"
msgstr ""

#: config/default.py:488
msgid "导入仪表盘"
msgstr ""

#: config/default.py:495
msgid "管理"
msgstr ""

#: config/default.py:501
msgid "日志接入"
msgstr ""

#: config/default.py:504
msgid "接入"
msgstr ""

#: config/default.py:508
msgid "日志采集"
msgstr ""

#: config/default.py:532
msgid "日志清洗"
msgstr ""

#: config/default.py:535
msgid "清洗"
msgstr ""

#: config/default.py:539
msgid "清洗列表"
msgstr ""

#: config/default.py:554 config/default.py:593 config/default.py:596
msgid "日志归档"
msgstr ""

#: config/default.py:561
msgid "归档仓库"
msgstr ""

#: config/default.py:567
msgid "归档列表"
msgstr ""

#: config/default.py:573
msgid "归档回溯"
msgstr ""

#: config/default.py:581
msgid "日志提取"
msgstr ""

#: config/default.py:583
msgid "提取"
msgstr ""

#: config/default.py:586
msgid "日志提取配置"
msgstr ""

#: config/default.py:588
msgid "提取链路管理"
msgstr ""

#: config/default.py:600
msgid "全链路追踪"
msgstr ""

#: config/default.py:620
msgid "SDK接入"
msgstr ""

#: config/default.py:625
msgid "ES集群"
msgstr ""

#: config/default.py:629
msgid "集群管理"
msgstr ""

#: config/default.py:633 config/default.py:636
msgid "设置"
msgstr ""

#: config/default.py:638
msgid "采集链路管理"
msgstr ""

#: version_log/config.py:31
msgid "版本日志"
msgstr ""

#: version_log/models.py:43
msgid "访问版本"
msgstr ""

#: version_log/models.py:48
msgid "版本日志访问记录"
msgstr ""

#: version_log/views.py:56
msgid "访问出错，请联系管理员。"
msgstr ""

#: version_log/views.py:57
msgid "日志列表获取成功"
msgstr ""

#: version_log/views.py:67
msgid "日志版本文件没找到，请联系管理员"
msgstr ""

#: version_log/views.py:69
msgid "日志详情获取成功"
msgstr ""
